# Hungarian translation of anaconda.
# This file is distributed under the same license as the anaconda package.
# Copyright (C) 2004-2009 Fedora Project.
# Arpad Biro <biro_arpad@yahoo.com>, 2004, 2005, 2006, 2007.
# Széll Tamás <tomi@digiflex.hu>, 2004.
# Gabor SUVEG <gabor@suveg.hu>, 2004.
# Szentiványi Gábor <szentivanyi.gabor@ulx.hu>, 2006.
# Gábor Szentiványi <szenti@ulx.hu>, 2006.
# Sulyok Péter <peti@fedoraproject.org>, 2008, 2009.
# Hoppár Zoltán <hopparz@gmail.com>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: anaconda master\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2009-09-17 14:51-1000\n"
"PO-Revision-Date: 2009-09-18 21:59+0100\n"
"Last-Translator: Zoltan Hoppar <hopparz@gmail.com>\n"
"Language-Team: Hungarian <Hungarian <fedora-trans-hu@redhat.com>>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: anaconda:374
msgid "Press <enter> for a shell"
msgstr "Parancsértelmezőért nyomjon <Enter>-t"

#: anaconda:389
#: rescue.py:324
#: rescue.py:350
#: rescue.py:363
#: rescue.py:439
#: rescue.py:451
#: text.py:545
#: loader/cdinstall.c:228
#: loader/cdinstall.c:232
#: loader/cdinstall.c:248
#: loader/cdinstall.c:251
#: loader/cdinstall.c:445
#: loader/cdinstall.c:452
#: loader/cdinstall.c:517
#: loader/dirbrowser.c:145
#: loader/driverdisk.c:98
#: loader/driverdisk.c:243
#: loader/driverdisk.c:274
#: loader/driverdisk.c:305
#: loader/driverdisk.c:340
#: loader/driverdisk.c:359
#: loader/driverdisk.c:371
#: loader/driverdisk.c:379
#: loader/driverdisk.c:539
#: loader/driverdisk.c:576
#: loader/driverselect.c:81
#: loader/driverselect.c:205
#: loader/hdinstall.c:134
#: loader/hdinstall.c:279
#: loader/hdinstall.c:340
#: loader/hdinstall.c:372
#: loader/hdinstall.c:442
#: loader/hdinstall.c:485
#: loader/hdinstall.c:498
#: loader/kbd.c:119
#: loader/kickstart.c:132
#: loader/kickstart.c:142
#: loader/kickstart.c:185
#: loader/kickstart.c:284
#: loader/kickstart.c:381
#: loader/kickstart.c:562
#: loader/lang.c:115
#: loader/lang.c:379
#: loader/loader.c:403
#: loader/loader.c:439
#: loader/loader.c:482
#: loader/loader.c:503
#: loader/loader.c:520
#: loader/loader.c:557
#: loader/loader.c:1190
#: loader/loader.c:1374
#: loader/mediacheck.c:46
#: loader/mediacheck.c:85
#: loader/mediacheck.c:92
#: loader/mediacheck.c:101
#: loader/method.c:279
#: loader/method.c:353
#: loader/modules.c:335
#: loader/modules.c:347
#: loader/net.c:574
#: loader/net.c:945
#: loader/net.c:1579
#: loader/net.c:1600
#: loader/net.c:1858
#: loader/net.c:1880
#: loader/net.c:1892
#: loader/nfsinstall.c:91
#: loader/nfsinstall.c:250
#: loader/nfsinstall.c:269
#: loader/nfsinstall.c:337
#: loader/telnetd.c:92
#: loader/urlinstall.c:83
#: loader/urlinstall.c:107
#: loader/urlinstall.c:175
#: loader/urlinstall.c:451
#: loader/urlinstall.c:460
#: loader/urlinstall.c:469
#: loader/urls.c:292
#: loader/urls.c:328
#: loader/urls.c:334
#: loader/urls.c:346
#: textw/constants_text.py:44
msgid "OK"
msgstr "OK"

#: anaconda:396
msgid "You do not have enough RAM to use the graphical installer.  Starting text mode."
msgstr "Nincs elég memória a grafikus telepítő használatához.  Telepítés szöveges módban."

#: anaconda:459
msgid "Starting graphical installation."
msgstr "Grafikus telepítő indul…"

#: anaconda:775
msgid "Would you like to use VNC?"
msgstr "Szeretne VNC-t használni?"

#: anaconda:776
msgid "Text mode provides a limited set of installation options.  It does not allow you to specify your own partitioning layout or package selections.  Would you like to use VNC mode instead?"
msgstr "A szöveges mód lekorlátozott telepítés opciókat kínál.  Nem engedi meg a saját merevlemez felosztást vagy a csomagok kiválogatását.  Kívánja inkább a VNC módot használni?"

#: anaconda:802
msgid "Install class forcing text mode installation"
msgstr "A telepítő osztály szöveges módú telepítést erőltet"

#: anaconda:821
msgid "Graphical installation is not available. Starting text mode."
msgstr "Grafikus telepítés nem érhető el. Szöveges mód indítása."

#: anaconda:829
msgid "DISPLAY variable not set. Starting text mode."
msgstr "DISPLAY változó nincs beállítva. Szöveges mód indítása."

#: anaconda:997
msgid "reipl configuration successful => reboot"
msgstr "sikeres reipl beállítás => újraindulás"

#: anaconda:1000
msgid "reipl configuration failed => halt"
msgstr "reipl beállítás sikertelen => leállás"

#: backend.py:148
#, python-format
msgid "Upgrading %s\n"
msgstr "%s frissítése\n"

#: backend.py:150
#, python-format
msgid "Installing %s\n"
msgstr "%s telepítése\n"

#: backend.py:170
msgid "Copying File"
msgstr "Fájl másolása"

#: backend.py:171
msgid "Transferring install image to hard drive"
msgstr "Telepítő képmás másolása a merevlemezre"

#: backend.py:181
msgid "An error occurred transferring the install image to your hard drive.  This is often cause by damaged or low quality media."
msgstr "A telepítő képmást merevlemezre másolva hiba történt.  Ezt gyakran sérült vagy gyenge minőségű adathordozó okozza."

#: backend.py:185
msgid "An error occurred transferring the install image to your hard drive. You are probably out of disk space."
msgstr "Hiba történt a telepítési képmás merevlemezre írása közben. Valószínűleg elfogyott a szabad hely."

#: backend.py:189
#: image.py:285
#: livecd.py:190
#: livecd.py:398
#: partIntfHelpers.py:237
#: text.py:308
#: text.py:312
#: yuminstall.py:378
#: yuminstall.py:722
#: yuminstall.py:836
#: yuminstall.py:1098
#: yuminstall.py:1162
#: yuminstall.py:1321
#: yuminstall.py:1339
#: iw/autopart_type.py:99
#: iw/autopart_type.py:198
#: iw/autopart_type.py:318
#: iw/autopart_type.py:321
#: iw/autopart_type.py:388
#: iw/autopart_type.py:396
#: iw/autopart_type.py:425
#: iw/osbootwidget.py:210
#: iw/osbootwidget.py:219
#: iw/raid_dialog_gui.py:707
#: iw/raid_dialog_gui.py:746
#: iw/task_gui.py:63
#: iw/task_gui.py:178
#: iw/task_gui.py:306
#: iw/task_gui.py:430
#: loader/cdinstall.c:228
#: loader/cdinstall.c:517
#: loader/driverdisk.c:98
#: loader/driverdisk.c:305
#: loader/driverdisk.c:340
#: loader/driverdisk.c:371
#: loader/driverdisk.c:379
#: loader/driverdisk.c:443
#: loader/hdinstall.c:134
#: loader/hdinstall.c:340
#: loader/hdinstall.c:442
#: loader/hdinstall.c:485
#: loader/hdinstall.c:498
#: loader/kickstart.c:284
#: loader/lang.c:115
#: loader/loader.c:403
#: loader/loader.c:520
#: loader/loader.c:1190
#: loader/mediacheck.c:46
#: loader/mediacheck.c:85
#: loader/mediacheck.c:92
#: loader/method.c:279
#: loader/method.c:353
#: loader/nfsinstall.c:250
#: loader/nfsinstall.c:269
#: loader/telnetd.c:92
#: loader/urlinstall.c:83
#: loader/urlinstall.c:107
#: loader/urlinstall.c:175
#: loader/urls.c:328
#: loader/urls.c:334
#: storage/__init__.py:190
#: storage/__init__.py:1631
#: storage/__init__.py:1716
#: textw/netconfig_text.py:280
#: textw/partition_text.py:186
#: textw/partition_text.py:192
#: textw/partition_text.py:198
#: textw/partition_text.py:231
#: textw/partition_text.py:241
#: textw/upgrade_text.py:175
#: textw/upgrade_text.py:182
msgid "Error"
msgstr "Hiba"

#: bootloader.py:53
#: bootloader.py:209
#: bootloader.py:216
#: image.py:82
#: upgrade.py:251
#: yuminstall.py:1502
#: yuminstall.py:1538
#: iw/autopart_type.py:208
#: iw/blpasswidget.py:148
#: iw/upgrade_swap_gui.py:191
#: iw/upgrade_swap_gui.py:199
#: iw/upgrade_swap_gui.py:206
#: storage/devicetree.py:143
#: storage/devicetree.py:170
#: textw/upgrade_text.py:187
msgid "Warning"
msgstr "Figyelmeztetés"

#: bootloader.py:54
msgid ""
"Filesystems have already been activated.  You cannot go back past this point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""
"A fájlrendszerek már működnek.  Ettől a ponttól folytatva, már nincs visszalépés.\n"
"\n"
"Szeretné folytatni a telepítéssel?"

#: bootloader.py:58
#: gui.py:1097
#: gui.py:1144
#: gui.py:1272
#: gui.py:1351
#: image.py:91
#: kickstart.py:151
#: livecd.py:197
#: livecd.py:405
#: packages.py:126
#: packages.py:134
#: packages.py:142
#: packages.py:150
#: packages.py:162
#: packages.py:171
#: packages.py:179
#: packages.py:188
#: upgrade.py:58
#: upgrade.py:206
#: yuminstall.py:235
#: yuminstall.py:655
#: yuminstall.py:840
#: yuminstall.py:920
#: yuminstall.py:926
#: yuminstall.py:1069
#: yuminstall.py:1094
#: yuminstall.py:1152
#: yuminstall.py:1316
#: yuminstall.py:1346
#: storage/__init__.py:91
#: storage/__init__.py:194
#: storage/__init__.py:1804
msgid "_Exit installer"
msgstr "_Kilépés a telepítőből"

#: bootloader.py:58
#: image.py:92
#: image.py:248
#: kickstart.py:1348
#: kickstart.py:1387
#: upgrade.py:58
#: yuminstall.py:1157
#: iw/partition_gui.py:1037
#: storage/__init__.py:1805
msgid "_Continue"
msgstr "_Tovább"

#: bootloader.py:114
#: platform.py:93
#: platform.py:319
#: platform.py:386
#: platform.py:486
msgid "RAID Device"
msgstr "RAID eszköz"

#: bootloader.py:143
msgid "Bootloader"
msgstr "Rendszerindító"

#: bootloader.py:143
msgid "Installing bootloader."
msgstr "Rendszerindító telepítése."

#: bootloader.py:210
msgid "There was an error installing the bootloader.  The system may not be bootable."
msgstr "Hiba történt a rendszerindító telepítése közben.  A rendszer valószínűleg nem lesz indítható."

#: bootloader.py:217
msgid "No kernel packages were installed on the system.  Bootloader configuration will not be changed."
msgstr "Nem települtek rendszermag csomagok. A rendszerindító beállításai nem változnak meg."

#: cmdline.py:55
msgid "Completed"
msgstr "Kész"

#: cmdline.py:63
msgid "In progress"
msgstr "Folyamatban"

#: cmdline.py:86
#: gui.py:1139
#: kickstart.py:1228
#: text.py:391
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration file:\n"
"\n"
"%s"
msgstr ""
"A következő hiba merült fel a kickstart konfigurációs fájl értelmezése közben:\n"
"\n"
"%s"

#: cmdline.py:98
msgid "Command line mode requires all choices to be specified in a kickstart configuration file."
msgstr "A parancssoros módhoz minden választást meg kell adni a kickstart konfigurációs fájlban."

#: cmdline.py:114
#: cmdline.py:121
msgid "Can't have a question in command line mode!"
msgstr "Nem lehet kérdezni parancssoros módban!"

#: constants.py:74
msgid "An unhandled exception has occurred.  This is most likely a bug.  Please save a copy of the detailed exception and file a bug report"
msgstr "Kezeletlen kivétel történt.  A legvalószínűbb, hogy ez programhiba.  Kérem, másolja ki a kivétel teljes szövegét, és adjon fel egy hibabejelentést"

#: constants.py:80
msgid " with the provider of this software."
msgstr " e szoftver szállítójának."

#: constants.py:84
#, python-format
msgid " against anaconda at %s"
msgstr " anaconda tárggyal %s helyen"

#: gui.py:106
msgid "An error occurred saving screenshots to disk."
msgstr "A képernyőképek lemezre mentése közben hiba történt."

#: gui.py:117
msgid "Screenshots Copied"
msgstr "Képernyőképek átmásolva"

#: gui.py:118
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"A képernyőképek a következő mappába kerültek:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"Újraindítás után rendszergazdaként belépve érheti el."

#: gui.py:161
msgid "Saving Screenshot"
msgstr "Képernyőkép mentése"

#: gui.py:162
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "„%s” nevű képernyőkép elmentve."

#: gui.py:165
msgid "Error Saving Screenshot"
msgstr "A képernyőkép mentése közben hiba történt"

#: gui.py:166
msgid "An error occurred while saving the screenshot.  If this occurred during package installation, you may need to try several times for it to succeed."
msgstr "Képernyőkép mentése közben hiba történt.  Ha ez a csomag telepítése alatt történt, akkor lehet, hogy többször is meg kell próbálni, hogy sikeres legyen."

#: gui.py:560
#: text.py:348
msgid "Installation Key"
msgstr "Telepítő kulcs"

#: gui.py:631
#: text.py:148
msgid "Choose a passphrase for the encrypted devices. You will be prompted for this passphrase during system boot."
msgstr "Válasszon jelszót a titkosított eszközöknek. A rendszerindító induláskor kérni fogja ezt a jelszót."

#: gui.py:650
#: gui.py:658
#: text.py:186
#: text.py:196
msgid "Error with passphrase"
msgstr "Jelszóhiba"

#: gui.py:651
#: text.py:187
msgid "The passphrases you entered were different.  Please try again."
msgstr "A megadott jelszók különböznek.  Kérem próbálja újra."

#: gui.py:659
msgid "The passphrase must be at least eight characters long."
msgstr "A jelszó legalább 8 jel hosszú kell legyen."

#: gui.py:693
#: text.py:224
#, python-format
msgid "Device %s is encrypted. In order to access the device's contents during installation you must enter the device's passphrase below."
msgstr "%s eszköz titkosítva. Ahhoz, hogy hozzáférjek az eszköz tartalmához telepítés közben, önnek alább meg kell adnia az eszköz jelszavát."

#: gui.py:784
#: gui.py:1351
#: partIntfHelpers.py:157
#: partIntfHelpers.py:339
#: text.py:99
#: text.py:100
#: iw/account_gui.py:130
#: loader/dirbrowser.c:145
#: loader/driverdisk.c:244
#: loader/kickstart.c:381
#: loader/loader.c:439
#: loader/loader.c:557
#: textw/constants_text.py:48
msgid "Cancel"
msgstr "Mégsem"

#: gui.py:796
#: gui.py:797
#: gui.py:913
#: gui.py:914
#: ui/anaconda.glade.h:2
msgid "_Debug"
msgstr "Ny_omkövetés"

#: gui.py:1141
#: text.py:393
msgid "Error Parsing Kickstart Config"
msgstr "Hiba történt a Kickstart beállítás feldolgozásakor"

#: gui.py:1181
msgid "default:LTR"
msgstr "alapértelmezés:LTR"

#: gui.py:1261
#: text.py:510
msgid "Error!"
msgstr "Hiba!"

#: gui.py:1262
#: text.py:511
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""
"Telepítő felülelet komponensét betöltvén hiba történt.\n"
"\n"
"osztály = %s"

#: gui.py:1267
#: image.py:164
#: image.py:194
#: packages.py:424
#: packages.py:429
#: storage/__init__.py:1630
msgid "_Exit"
msgstr "_Kilépés"

#: gui.py:1268
#: image.py:164
#: image.py:194
#: livecd.py:197
#: yuminstall.py:714
#: yuminstall.py:1094
#: yuminstall.py:1152
#: yuminstall.py:1316
msgid "_Retry"
msgstr "Újr_a"

#: gui.py:1271
msgid "The installer will now exit."
msgstr "A telepítő most kilép."

#: gui.py:1274
#: storage/partitioning.py:267
msgid "The system will now reboot."
msgstr "A rendszer most újraindul."

#: gui.py:1275
#: image.py:248
#: packages.py:432
#: yuminstall.py:1362
msgid "_Reboot"
msgstr "Újrai_ndítás"

#: gui.py:1277
msgid "Exiting"
msgstr "Kilépés"

#: gui.py:1348
#: livecd.py:115
#: text.py:266
#: upgrade.py:197
msgid "Exit installer"
msgstr "Kilépés a telepítőből"

#: gui.py:1349
msgid "Are you sure you wish to exit the installer?"
msgstr "Biztos, hogy ki kíván lépni a telepítőből?"

#: gui.py:1358
#, python-format
msgid "%s Installer"
msgstr "%s telepítő"

#: gui.py:1364
msgid "Unable to load title bar"
msgstr "Nem sikerült betölteni a címsort"

#: gui.py:1426
msgid "Install Window"
msgstr "Telepítőablak"

#: image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can choose to continue if you think this is in error."
msgstr ""
"%s ISO képmás mérete nem 2048 bájt többszöröse.  Ez azt is jelentheti, hogy erre a számítógépre áthozva megsérült a képmás.\n"
"\n"
"Ajánlom, hogy lépjen ki, és szakítsa meg a telepítést. Azonban választhatja a folytatást is, ha úgy gondolja, hogy ez egy hiba."

#: image.py:156
msgid "Couldn't Mount ISO Source"
msgstr "Nem sikerült ISO forrást becsatolni"

#: image.py:157
#, python-format
msgid "An error occurred mounting the source device %s.  This may happen if your ISO images are located on an advanced storage device like LVM or RAID, or if there was a problem mounting a partition.  Click exit to abort the installation."
msgstr "%s forráseszköz becsatolása közben hiba történt.  Ez megtörténhet, ha az ISO képmások egy fejlett tároló eszközön vannak (mint LVM vagy RAID), vagy ha baj volt a lemezrész becsatolásával.  Kattintson a kilépésre, hogy kilépjen a telepítőből."

#: image.py:185
msgid "Missing ISO 9660 Image"
msgstr "Hiányzó ISO 9660 képmás"

#: image.py:186
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort the installation."
msgstr ""
"A telepítő megpróbálta becsatolni #%s képet, de nem találja a merevlemezen.\n"
"\n"
"Kérem másolja ezt a képet a merevlemezre, és kattintson az Újra gombra.  Kattintson a Kilépésre, hogy megszakítsa a telepítést."

#: image.py:241
msgid "Required Install Media"
msgstr "Szükséges telepítő adathordozók"

#: image.py:242
#, python-format
msgid ""
"The software you have selected to install will require the following %s %s discs:\n"
"\n"
"%s\n"
"Please have these ready before proceeding with the installation.  If you need to abort the installation and exit please select \"Reboot\"."
msgstr ""
"A kiválasztott szoftverek telepítéséhez a következő %s %s lemezek \n"
"szükségesek:\n"
"\n"
"%s\n"
"Kérem készítse elő őket, mielőtt folytatná a telepítést.  Ha meg kell szakítania a telepítést, válassza az „Újraindítás” gombot."

#: image.py:248
#: livecd.py:404
#: packages.py:378
#: packages.py:429
#: packages.py:432
#: upgrade.py:205
#: yuminstall.py:840
#: yuminstall.py:926
#: yuminstall.py:1346
#: yuminstall.py:1362
msgid "_Back"
msgstr "_Vissza"

#: image.py:286
#, python-format
msgid "An error occurred unmounting the disc.  Please make sure you're not accessing %s from the shell on tty2 and then click OK to retry."
msgstr "Lemez lecsatolása közben hiba történt.  Kérem győződjön meg arról, hogy nem használja %s eszközt egy parancsértelmezőből a tty2-n, majd kattintson az OK gombra, hogy újra megpróbálja."

#: installclass.py:74
msgid "Install on System"
msgstr "Telepítés a rendszerre"

#: iutil.py:831
#, python-format
msgid "Error: On open, cannot set reIPL method to %s (%s: %s)"
msgstr "Hiba: Nyitáskor nem lehet reIPL eljárást a következőre állítani: %s (%s: %s)"

#: iutil.py:839
#, python-format
msgid "Error: On write, cannot set reIPL method to %s (%s: %s)"
msgstr "Hiba: Íráskor nem lehet reIPL eljárást a következőre állítani: %s (%s: %s)"

#: iutil.py:846
#, python-format
msgid "Error: On close, cannot set reIPL method to %s (%s: %s)"
msgstr "Hiba: Záráskor nem lehet reIPL eljárást a következőre állítani: %s (%s: %s)"

#: iutil.py:863
#, python-format
msgid "Error: Could not set %s as reIPL device (%s)"
msgstr "Hiba: Nem lehet %s eszközt reIPL-nek beállítani (%s)"

#: iutil.py:872
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr "Hiba: Nem lehet loadparm-ot törölni (%s)"

#: iutil.py:881
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr "Figyelem: Nem lehet parm-ot törölni (%s)"

#: iutil.py:891
#, python-format
msgid "After shutdown, please perform a manual IPL from DASD device %s to continue installation"
msgstr "Leállás után, kérem kézivezérléssel végezze el az IPL-t %s DASD eszközről, hogy folytassa a telepítést"

#: iutil.py:913
#, python-format
msgid "Error: reading FCP property %s for reIPL (%s)"
msgstr "Hiba: %s FCP tulajdonságot reIPL számára beolvasása közben (%s)"

#: iutil.py:926
#, python-format
msgid "Error: writing FCP property %s for reIPL (%s)"
msgstr "Hiba: %s FCP tulajdonságot reIPL számára kiírása közben (%s)"

#: iutil.py:939
#, python-format
msgid "Error: writing default FCP property %s for reIPL (%s)"
msgstr "Hiba: %s FCP gyári tulajdonságot reIPL számára kiírása közben (%s)"

#: iutil.py:949
#, python-format
msgid "After shutdown, please perform a manual IPL from FCP %(device)s with WWPN %(wwpn)s and LUN %(lun)s to continue installation"
msgstr "Leállás után, a telepítés folytatásához kérem végezze el kézivezérléssel az IPL-t FCP %(device)s eszközről WWPN %(wwpn)s és LUN %(lun)s tulajdonságokkal"

#: iutil.py:956
msgid "After shutdown, please perform a manual IPL from the device now containing /boot to continue installation"
msgstr "Leállás után, a telepítés folytatásához kérem végezzen IPL-t kézivezérléssel a /boot eszközéről"

#: iutil.py:964
msgid "Error determining mount point type"
msgstr "Hiba a csatolási pont típusának meghatározása közben"

#: iutil.py:968
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr "A / vagy a /boot csatolási pont számunkra ismeretlen lemezen van"

#: kickstart.py:97
#, python-format
msgid "There was an error running the kickstart script at line %s.  You may examine the output in %s.  This is a fatal error and installation will be aborted.  Press the OK button to exit the installer."
msgstr "A kickstart szkript feldolgozása alatt a %s. sorban hiba történt.  %s fájlban megvizsgálhatja a kimenetet.  Ez végzetes hiba és  a telepítés nem  folytatható.  Nyomjon OK-t, a telepítőből való kilépéshez."

#: kickstart.py:112
#: kickstart.py:114
msgid "Scriptlet Failure"
msgstr "Kickstart szkript hiba"

#: kickstart.py:145
#: rescue.py:240
#: yuminstall.py:650
#: yuminstall.py:1064
msgid "No Network Available"
msgstr "Nincs elérhető hálózat"

#: kickstart.py:146
msgid "Encryption key escrow requires networking, but there was an error enabling the network on your system."
msgstr "Titkosítási kulcs zárolásához hálózat kell, de a hálózat engedélyezése közben hiba történt ."

#: kickstart.py:1214
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr "Hiba %%ksappend sorok feldolgozása alatt: %s"

#: kickstart.py:1217
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr "Ismeretlen hiba %%ksappend sorok feldolgozása alatt: %s"

#: kickstart.py:1289
#: livecd.py:221
msgid "Post-Installation"
msgstr "Zárótelepítés"

#: kickstart.py:1290
msgid "Running post-installation scripts"
msgstr "Telepítést záró szkriptek futtatása"

#: kickstart.py:1306
msgid "Pre-Installation"
msgstr "Telepítés előkészítés"

#: kickstart.py:1307
msgid "Running pre-installation scripts"
msgstr "Telepítés előkészítési szkriptek futtatása"

#: kickstart.py:1339
msgid "Missing Package"
msgstr "Hiányzó csomag"

#: kickstart.py:1340
#, python-format
msgid "You have specified that the package '%s' should be installed.  This package does not exist. Would you like to continue or abort this installation?"
msgstr "„%s” csomagot megjelölte telepítésre.  Nincs ilyen csomag. Folytatni akarja, vagy megszakítja a telepítést?"

#: kickstart.py:1346
#: kickstart.py:1385
msgid "_Abort"
msgstr "_Megszakítás"

#: kickstart.py:1347
#: kickstart.py:1386
msgid "_Ignore All"
msgstr "_Mindegyik mellőzése"

#: kickstart.py:1377
msgid "Missing Group"
msgstr "Hiányzó csoport"

#: kickstart.py:1378
#, python-format
msgid "You have specified that the group '%s' should be installed. This group does not exist. Would you like to continue or abort this installation?"
msgstr "„%s” csoportot megjelölte telepítésre. Nincs ilyen csoport. Folytatni akarja, vagy megszakítja a telepítést?"

#: kickstart.py:1484
#, python-format
msgid ""
"The kickstart configuration file is missing required information that anaconda cannot prompt for.  Please add the following sections and try again:\n"
"%s"
msgstr ""
"A kickstart beállítás fájlból olyan adat hiányzik, amit a telepítő nem tud bekérni.  Kérem adja hozzá a következő szakaszokat, majd próbálja újra:\n"
"%s"

#: livecd.py:110
msgid "Unable to find image"
msgstr "Képmás nem található"

#: livecd.py:111
#, python-format
msgid "The given location isn't a valid %s live CD to use as an installation source."
msgstr "A megadott hely nem érvényes telepítési forrásként használható %s élő CD."

#: livecd.py:172
msgid "Copying live image to hard drive."
msgstr "Élő képmás másolása a merevlemezre."

#: livecd.py:191
msgid ""
"There was an error installing the live image to your hard drive.  This could be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will require reinstallation."
msgstr ""
"Hiba történt az élő kép merevlemezre másolása közben.  Talán sérült hordozó okozta.  Kérem ellenőrizze a telepítő hordozót.\n"
"\n"
"Ha kilép, a rendszer töredékkel teli marad, s emiatt újbóli telepítést igényel."

#: livecd.py:222
msgid "Performing post-installation filesystem changes.  This may take several minutes."
msgstr "Fájlrendszer változásainak zárótelepítése.  Ez néhány percig eltarthat."

#: livecd.py:399
#, python-format
msgid "The root filesystem you created is not large enough for this live image (%.2f MB required)."
msgstr "A gyökér fájlrendszer, amit ön készített nem elég nagy ehhez az élő képhez (%.2f·MB·szükséges)."

#: network.py:56
msgid "Hostname must be 255 or fewer characters in length."
msgstr "A gazdanév legfeljebb 255 jelből állhat."

#: network.py:62
msgid "Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or '0-9'"
msgstr "A gazda név első betűje az „a-z”, „A-Z” vagy „0-9” intervallumban kell legyen."

#: network.py:67
msgid "Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr "A gazda nevében kizárólag engedélyezett jelek: 'a-z', 'A-Z', '0-9', '-', vagy '.'"

#: network.py:176
msgid "IP address is missing."
msgstr "IP cím hiányzik."

#: network.py:180
msgid "IPv4 addresses must contain four numbers between 0 and 255, separated by periods."
msgstr "IPv4 cím 4 darab, 0 és 255 közé eső, egymástól ponttal elválasztott számot kell tartalmazzon."

#: network.py:183
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "„%s” nem érvényes IPv6 cím."

#: network.py:185
#, python-format
msgid "'%s' is an invalid IP address."
msgstr "„%s” érvénytelen IP cím."

#: packages.py:121
msgid "Device Resize Failed"
msgstr "Az eszköz átméretezése sikertelen"

#: packages.py:122
#, python-format
msgid "An error was encountered while resizing device %s."
msgstr "%s eszköz átméretezése alatt hiba történt."

#: packages.py:129
msgid "Device Creation Failed"
msgstr "Eszköz elkészítése sikertelen"

#: packages.py:130
#, python-format
msgid "An error was encountered while creating device %s."
msgstr "%s eszköz elkészítése közben hiba történt."

#: packages.py:137
msgid "Device Removal Failed"
msgstr "Eszköz eltávolítása sikertelen"

#: packages.py:138
#, python-format
msgid "An error was encountered while removing device %s."
msgstr "%s eszköz eltávolítása közben hiba történt."

#: packages.py:145
msgid "Device Setup Failed"
msgstr "Eszköz beállítása sikertelen"

#: packages.py:146
#, python-format
msgid "An error was encountered while setting up device %s."
msgstr "%s eszköz beállítása közben hiba történt."

#: packages.py:157
msgid "Resizing Failed"
msgstr "Átméretezés sikertelen"

#: packages.py:158
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr "%s eszköz átméretezése közben hiba történt."

#: packages.py:165
msgid "Migration Failed"
msgstr "A migráció sikertelen"

#: packages.py:166
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr "A fájlrendszert átmozgatása közben hiba történt a %s eszközön."

#: packages.py:174
msgid "Formatting Failed"
msgstr "A formázás sikertelen"

#: packages.py:175
#, python-format
msgid "An error was encountered while formatting device %s."
msgstr "%s eszköz formázása közben hiba lépett fel."

#: packages.py:183
msgid "Storage Activation Failed"
msgstr "Tárolóeszköz aktiválása sikertelen"

#: packages.py:184
msgid "An error was encountered while activating your storage configuration."
msgstr "A tároló beállításainak érvényesítése közben hiba lépett fel."

#: packages.py:188
msgid "_File Bug"
msgstr "_Hiba iktatása"

#: packages.py:349
msgid "Invalid Key"
msgstr "Érvénytelen kulcs"

#: packages.py:350
msgid "The key you entered is invalid."
msgstr "A megadott kulcs érvénytelen."

#: packages.py:375
#: rescue.py:287
#: rescue.py:289
#: loader/cdinstall.c:248
#: loader/cdinstall.c:251
#: loader/method.c:326
#: storage/__init__.py:1630
#: textw/upgrade_text.py:133
msgid "Skip"
msgstr "Kihagyás"

#: packages.py:378
msgid "_Skip"
msgstr "K_ihagyás"

#: packages.py:410
#: packages.py:433
msgid "Warning! This is pre-release software!"
msgstr "Figyelmeztetés! Ez kiadás előtti szoftver!"

#: packages.py:411
#, python-format
msgid ""
"Thank you for downloading this pre-release of %s.\n"
"\n"
"This is not a final release and is not intended for use on production systems.  The purpose of this release is to collect feedback from testers, and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %s\n"
"\n"
"and file a report against '%s'.\n"
msgstr ""
"Köszönjük, hogy letöltötte %s rendszer előzetes kiadását.\n"
"\n"
"Ez még nem a végleges változat, nem ajánljuk használatát éles rendszerben.  Azért van, hogy visszajelzéseket kapjunak tesztelőktől, és nem való mindennapos használatra.\n"
"\n"
"A visszajelzéseket ide kérjük:\n"
"\n"
"   %s\n"
"\n"
"Kérjük, hogy a hibákat „%s” címen jelentse be.\n"

#: packages.py:424
msgid "_Install anyway"
msgstr "_Telepítés mindenképpen"

#: packages.py:428
msgid "The installer will now exit..."
msgstr "A telepítő most kilép..."

#: packages.py:431
msgid "Your system will now be rebooted..."
msgstr "A rendszer újraindítása következik..."

#: partIntfHelpers.py:41
msgid "Please enter a volume group name."
msgstr "Kérem adjon meg egy kötetcsoportnevet."

#: partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr "Kötet csoport név kevesebb, mint 128 jel kell legyen"

#: partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "Hiba történt – %s kötet csoport név érvénytelen."

#: partIntfHelpers.py:53
msgid "Error - the volume group name contains illegal characters or spaces.  Acceptable characters are letters, digits, '.' or '_'."
msgstr "Hiba – a kötet csoport név érvénytelen jeleket vagy szóközöket tartalmaz. Csak ékezetlen betű, számjegy, „.” és „_” használható."

#: partIntfHelpers.py:63
msgid "Please enter a logical volume name."
msgstr "Kérem adjon meg logikai kötet nevet."

#: partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr "Logikai kötet név 128 jelnél rövidebb kell legyen"

#: partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "Hiba – %s logikai kötet név érvénytelen."

#: partIntfHelpers.py:77
msgid "Error - the logical volume name contains illegal characters or spaces.  Acceptable characters are letters, digits, '.' or '_'."
msgstr "Hiba – a logikai kötet név érvénytelen jelet vagy szóközt tartalmaz.  Csak ékezetlen betű, számjegy, „.” vagy „_” használható."

#: partIntfHelpers.py:101
#, python-format
msgid "The mount point %s is invalid.  Mount points must start with '/' and cannot end with '/', and must contain printable characters and no spaces."
msgstr "%s csatoló pont érvénytelen.  Csatoló pontnak „/” jellel kell kezdődnie, nem végződhet „/” jelre, és csak nyomtatható jeleket tartalmazhat, szóközöket nem."

#: partIntfHelpers.py:108
msgid "Please specify a mount point for this partition."
msgstr "Kérem adjon meg csatolási pontot ehhez a partícióhoz."

#: partIntfHelpers.py:121
#: partIntfHelpers.py:128
msgid "Unable To Delete"
msgstr "Nem sikerült törölni"

#: partIntfHelpers.py:122
msgid "You must first select a partition to delete."
msgstr "Először ki kell választani a törölni kívánt partíciót."

#: partIntfHelpers.py:153
#: partIntfHelpers.py:338
#: iw/lvm_dialog_gui.py:813
msgid "Confirm Delete"
msgstr "Törlés megerősítése"

#: partIntfHelpers.py:154
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "„%s” eszközön minden részt törölni fog."

#: partIntfHelpers.py:157
#: partIntfHelpers.py:339
#: iw/lvm_dialog_gui.py:816
#: iw/lvm_dialog_gui.py:1349
#: iw/osbootwidget.py:103
#: iw/partition_gui.py:1337
msgid "_Delete"
msgstr "Tö_rlés"

#: partIntfHelpers.py:204
msgid "Notice"
msgstr "Megjegyzés"

#: partIntfHelpers.py:205
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"A következő részeket nem töröltem, mert használják:\n"
"\n"
"%s"

#: partIntfHelpers.py:220
msgid "Format as Swap?"
msgstr "Kialakítás lapozóként?"

#: partIntfHelpers.py:221
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"%s partíció típusa 0x82 (Linux lapozó), de úgy tűnik, nem aszerint alakították ki.\n"
"\n"
"Szeretné ezt a partíciót lapozó részként kialakítani?"

#: partIntfHelpers.py:236
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr "Legalább egy merevlemezt ki kell választani %s telepítéséhez."

#: partIntfHelpers.py:241
msgid "You have chosen to use a pre-existing partition for this installation without formatting it. We recommend that you format this partition to make sure files from a previous operating system installation do not cause problems with this installation of Linux. However, if this partition contains files that you need to keep, such as home directories, then continue without formatting this partition."
msgstr "Egy már meglévő részt választott ki telepítésre megújítás nélkül. Ajánljuk, hogy új fájlrendszert alakítson ki e részen, hogy az új rendszer működésében biztosan ne okozzanak bajt a korábbi rendszerhez tartozó fájlok. Azonban, ha ezen a részen olyan fájlok vannak, amelyeket meg szeretne őrizni, például a használók saját könyvtárai, akkor folytassa megújítás nélkül."

#: partIntfHelpers.py:249
msgid "Format?"
msgstr "Formázás?"

#: partIntfHelpers.py:249
#: iw/partition_gui.py:1035
msgid "_Modify Partition"
msgstr "Partíció _módosítása"

#: partIntfHelpers.py:249
msgid "Do _Not Format"
msgstr "Átalakítás _nélkül"

#: partIntfHelpers.py:257
msgid "Error with Partitioning"
msgstr "Hiba a particionálásnál"

#: partIntfHelpers.py:258
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. These errors must be corrected prior to continuing with your install of %s.\n"
"\n"
"%s"
msgstr ""
"A kívánt felosztási sémában kritikus hibák a következők. Ezeket a hibákat ki kell javítani, mielőtt folytatná %s telepítését.\n"
"\n"
"%s"

#: partIntfHelpers.py:272
msgid "Partitioning Warning"
msgstr "Particionálási figyelmeztetés"

#: partIntfHelpers.py:273
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"A következő figyelmeztetések keletkeztek a kért felosztási sémához.\n"
"\n"
"%s\n"
"\n"
"Szeretné folytatni a kért felosztó sémával?"

#: partIntfHelpers.py:287
#: iw/partition_gui.py:639
msgid "The following pre-existing partitions have been selected to be formatted, destroying all data."
msgstr "A következő korábban már kialakított partíciókat jelölte ki formázásra az összes adat törlésével."

#: partIntfHelpers.py:290
msgid "Select 'Yes' to continue and format these partitions, or 'No' to go back and change these settings."
msgstr "Válassza az „Igen”-t a folytatáshoz és a partíciók formázásához, vagy a „Nem”-et a visszalépéshez és a beállítások megváltoztatásához."

#: partIntfHelpers.py:296
msgid "Format Warning"
msgstr "Formázási figyelmeztetés"

#: partIntfHelpers.py:322
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"„%s” kötet csoport törlésére készül.\n"
"\n"
"A kötet csoporthoz tartozó összes logikai kötet EL FOG VESZNI!"

#: partIntfHelpers.py:326
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "„%s” logikai kötet törlésére készül."

#: partIntfHelpers.py:329
msgid "You are about to delete a RAID device."
msgstr "RAID eszköz törlésére készül."

#: partIntfHelpers.py:331
#, python-format
msgid "You are about to delete the %s partition."
msgstr "%s rész törlésére készül."

#: partIntfHelpers.py:335
#, python-format
msgid "You are about to delete the %s %s"
msgstr "%s %s törlésére készül."

#: partIntfHelpers.py:346
msgid "Confirm Reset"
msgstr "Visszaállítás megerősítése"

#: partIntfHelpers.py:347
msgid "Are you sure you want to reset the partition table to its original state?"
msgstr "Biztosan vissza akarja állítani a partíciós tábla eredeti állapotát?"

#: platform.py:94
#: platform.py:97
#: platform.py:320
#: platform.py:387
#: platform.py:487
#: platform.py:490
msgid "Master Boot Record (MBR)"
msgstr "Master Boot Record (MBR)"

#: platform.py:96
#: platform.py:489
msgid "First sector of boot partition"
msgstr "A rendszerindító rész első szektora"

#: platform.py:112
msgid "You have not created a bootable partition."
msgstr "Nem csinált rendszerindító részt."

#: platform.py:115
msgid "Bootable partitions can only be on RAID1 devices."
msgstr "Rendszerindító partíció csak RAID1 eszközön lehet."

#: platform.py:119
msgid "Bootable partitions cannot be on a logical volume."
msgstr "Rendszerindító partíció nem lehet logikai köteten."

#: platform.py:123
msgid "Bootable partitions cannot be on a RAID device."
msgstr "Rendszerindító partíció nem lehet RAID eszközön."

#: platform.py:127
#: platform.py:131
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr "Rendszerindító partíció nem lehet %s fájlrendszeren."

#: platform.py:135
#: platform.py:140
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr "Rendszerindító partíció nem lehet titkosított eszközön."

#: platform.py:214
msgid "EFI System Partition"
msgstr "EFI rendszer partíció"

#: platform.py:219
msgid "You have not created a /boot/efi partition."
msgstr "Nem csinált /boot/efi részt."

#: platform.py:225
msgid "/boot is not on an ext2 filesystem."
msgstr "/boot nem ext2 fájlrendszer."

#: platform.py:228
msgid "/boot/efi is not EFI."
msgstr "/boot/efi nem EFI."

#: platform.py:265
#, python-format
msgid "%s must have a bsd disk label."
msgstr "%s résznek rendelkeznie kell BSD lemez címkével."

#: platform.py:277
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr "%s lemez elején legalább 1MB szabad helynek kell lennie."

#: platform.py:322
msgid "PPC PReP Boot"
msgstr "PPC PReP rendszerindító"

#: platform.py:334
msgid "The boot partition must be within the first 4MB of the disk."
msgstr "A boot résznek a lemez első 4MB-jában kell lennie."

#: platform.py:389
#: platform.py:392
msgid "Apple Bootstrap"
msgstr "Apple rendszerindító"

#: platform.py:406
#, python-format
msgid "%s must have a mac disk label."
msgstr "%s résznek kell egy mac lemez címke."

#: rescue.py:199
msgid "When finished please exit from the shell and your system will reboot."
msgstr "Amint elkészült, kérem lépjen ki a rendszerhéjból, és a rendszer újra fog indulni."

#: rescue.py:213
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr "Nem találom a futtatandó /bin/sh-t!  Nem indítok parancssort"

#: rescue.py:232
msgid "Setup Networking"
msgstr "Hálózat beállítása"

#: rescue.py:233
msgid "Do you want to start the network interfaces on this system?"
msgstr "El szeretné indítani a rendszer hálózati csatolóit?"

#: rescue.py:234
#: loader/driverdisk.c:483
#: loader/driverdisk.c:493
#: loader/hdinstall.c:222
#: textw/constants_text.py:56
msgid "Yes"
msgstr "Igen"

#: rescue.py:234
#: rescue.py:236
#: loader/driverdisk.c:483
#: loader/driverdisk.c:493
#: textw/constants_text.py:60
msgid "No"
msgstr "Nem"

#: rescue.py:241
msgid "Unable to activate a networking device.  Networking will not be available in rescue mode."
msgstr "Nem lehet a hálózati eszközt bekapcsolni.  Hálózat nem lesz elérhető mentő módban."

#: rescue.py:276
#: rescue.py:345
#: rescue.py:356
#: rescue.py:434
msgid "Rescue"
msgstr "Mentés"

#: rescue.py:277
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and mount it under the directory %s.  You can then make any changes required to your system.  If you want to proceed with this step choose 'Continue'.  You can also choose to mount your file systems read-only instead of read-write by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"A mentő környezet megpróbálja megtalálni a Linux rendszert, és becsatolja azt a %s mappába. Ez után elvégezhető a kívánt módosítás a rendszerben.  Ha ezt választja, nyomjon „Folytatás”-t.  „Csak olvasható”-t választva, csak olvasható módon csatolja be a fájlrendszert írható helyett.\n"
"\n"
"Ha valamilyen okból ez a művelet nem sikerül, válassza a „Kihagyás”-t, így közvetlenül a parancssorba fog jutni.\n"
"\n"

#: rescue.py:287
#: iw/partition_gui.py:543
#: loader/cdinstall.c:206
#: loader/cdinstall.c:214
#: loader/driverdisk.c:444
#: storage/__init__.py:136
#: storage/devicetree.py:83
msgid "Continue"
msgstr "Folytatás"

#: rescue.py:287
#: rescue.py:292
msgid "Read-Only"
msgstr "Csak olvasható"

#: rescue.py:321
msgid "System to Rescue"
msgstr "Mentendő rendszer"

#: rescue.py:322
msgid "Which device holds the root partition of your installation?"
msgstr "Melyik rész tartalmazza a meglévő rendszer gyökerét?"

#: rescue.py:324
#: rescue.py:328
#: text.py:515
#: text.py:517
msgid "Exit"
msgstr "Kilépés"

#: rescue.py:346
msgid "Your system had dirty file systems which you chose not to mount.  Press return to get a shell from which you can fsck and mount your partitions.  The system will reboot automatically when you exit from the shell."
msgstr "A rendszerben rendezetlen fájlrendszerek vannak, amiket úgy döntött, nem csatlakoztat.  Nyomjon Enter-t, hogy a parancssorban a részeket fsck-val kijavíthatja, és becsatolhatja. A rendszer magától újraindul, amikor kilép a rendszerhéjból."

#: rescue.py:357
#, python-format
msgid ""
"Your system has been mounted under %s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the root environment, run the command:\n"
"\n"
"\tchroot %s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""
"A rendszer becsatolva %s ponton.\n"
"\n"
"Nyomjon <Enter>-t rendszerhéj megnyitásához. Ha a rendszert gyökér környezetté szeretné tenni, adja ki a következő parancsot:\n"
"\n"
"\tchroot %s\n"
"\n"
"A rendszer magától újraindul, amikor kilép a rendszerhéjból."

#: rescue.py:435
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you exit from the shell."
msgstr ""
"A rendszer néhány része vagy egésze becsatolása alatt hiba történt. Néhány még becsatolva maradhat %s alatt.\n"
"\n"
"Rendszerhéj megnyitásához nyomjon <Enter>-t. A rendszerhéjból kilépve a gép magától újra fog indulni."

#: rescue.py:444
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Nincs semmilyen Linux partíciója.  Újraindulás.\n"

#: rescue.py:447
msgid "Rescue Mode"
msgstr "Mentő mód"

#: rescue.py:448
msgid "You don't have any Linux partitions. Press return to get a shell. The system will reboot automatically when you exit from the shell."
msgstr "Nincs Linux partíció. Nyomjon Entert a parancssorhoz. A rendszer magától újraindul, amikor kilép a parancssorból."

#: rescue.py:461
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "A rendszer becsatolva %s ponton."

#: text.py:153
msgid "Passphrase for encrypted device"
msgstr "Jelszó a titkosított részhez"

#: text.py:166
msgid "Also add this passphrase to all existing encrypted devices"
msgstr "E jelszó használata minden meglévő titkosított eszközhöz"

#: text.py:231
#: ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr "Jelszó"

#: text.py:239
#: ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr "Ez globális jelszó"

#: text.py:309
#: text.py:313
msgid "Repository editing is not available in text mode."
msgstr "Tároló szerkesztés nem érhető el szöveges módban."

#: text.py:353
#, python-format
msgid "Please enter your %(instkey)s"
msgstr "Kérem adja meg ezt: %(instkey)s"

#: text.py:370
#: ui/instkey.glade.h:6
#, no-c-format, python-format
msgid "Skip entering %(instkey)s"
msgstr "%(instkey)s megadásának kihagyása"

#: text.py:420
#: loader/lang.c:63
#: loader/loader.c:215
#, c-format, python-format
msgid "Welcome to %s for %s"
msgstr "Üdvözli %s %s"

#: text.py:422
#, python-format
msgid "Welcome to %s"
msgstr "Üdvözli %s"

#: text.py:424
msgid "  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next screen"
msgstr "<Tab>/<Alt-Tab> elemek közti váltás  |  <Szóköz>  kiválasztás  |  <F12> köv. képernyő"

#: text.py:515
#: loader/net.c:111
#: loader/net.c:464
#: loader/net.c:520
#: loader/net.c:663
#: loader/net.c:671
#: loader/net.c:1094
#: loader/net.c:1101
msgid "Retry"
msgstr "Újra"

#: text.py:541
msgid "Cancelled"
msgstr "Törölve"

#: text.py:542
msgid "I can't go to the previous step from here. You will have to try again."
msgstr "Innen nem lehet visszalépni. Újra meg kell próbálnia."

#: upgrade.py:51
msgid "Proceed with upgrade?"
msgstr "Frissítéssel folytatja?"

#: upgrade.py:52
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"A frissíteni kívánt Linux telepítés fájlrendszereit már csatlakoztattam. Ettől a ponttól már nem léphet vissza. \n"
"\n"

#: upgrade.py:56
msgid "Would you like to continue with the upgrade?"
msgstr "Folytatni szeretné a rendszer frissítését?"

#: upgrade.py:184
msgid "Mount failed"
msgstr "Becsatolás sikertelen"

#: upgrade.py:185
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""
"Az /etc/fstab fájlban felsorolt részeket becsatolva a következő hiba történt.  Kérem javítsa ki ezt a hibát, majd próbálkozzon a frissítéssel újra.\n"
"%s"

#: upgrade.py:193
#: upgrade.py:200
msgid "Upgrade root not found"
msgstr "Frissíthető gyökér nem található"

#: upgrade.py:194
msgid "The root for the previously installed system was not found."
msgstr "A korábban telepített rendszer root-ja nem található."

#: upgrade.py:201
msgid "The root for the previously installed system was not found.  You can exit installer or backtrack to choose installation instead of upgrade."
msgstr "Az előző rendszer gyökerét nem találtam.  Kiléphet a telepítőből, vagy visszaléphet, hogy a telepítést válassza a frissítés helyett."

#: upgrade.py:224
msgid ""
"The following files are absolute symbolic links, which we do not support during an upgrade. Please change them to relative symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"A következő fájlok abszolút szimbolikus linkek, melyeket a frissítési folyamat nem támogat. Kérem változtassa meg ezeket relatív szimbolikus linkekre, és indítsa újra a frissítési folyamatot.\n"
"\n"

#: upgrade.py:230
msgid "Absolute Symlinks"
msgstr "Abszolút szimbolikus linkek"

#: upgrade.py:241
msgid ""
"The following are directories which should instead be symbolic links, which will cause problems with the upgrade.  Please return them to their original state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"A következő mappák a várttal ellentétben nem szimbolikus linkek, ami gondot okoz frissítés  közben.  Kérem állítsa vissza az eredeti állapotukat szimbolikus linkekként, majd indítsa újra a frissítést.\n"
"\n"

#: upgrade.py:247
msgid "Invalid Directories"
msgstr "Érvénytelen mappák"

#: upgrade.py:252
#, python-format
msgid "%s not found"
msgstr "Nincs %s"

#: vnc.py:137
#, python-format
msgid "%s %s installation on host %s"
msgstr "%s %s telepítés %s gazdára"

#: vnc.py:139
#, python-format
msgid "%s %s installation"
msgstr "%s %s telepítés"

#: vnc.py:166
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Csatlakozás %s gazdán levő vnc ügyfélhez..."

#: vnc.py:180
msgid "Connected!"
msgstr "Kapcsolódva!"

#: vnc.py:183
msgid "Will try to connect again in 15 seconds..."
msgstr "Újabb próbálkozás 15 másodperc múlva..."

#: vnc.py:200
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr "Kérem csatlakozzon a vnc ügyféllel %s gazdához, hogy megkezdje a telepítést."

#: vnc.py:202
msgid "Please manually connect your vnc client to begin the install."
msgstr "Kérem csatlakozzon vnc ügyféllel, hogy megkezdje a telepítést."

#: vnc.py:205
msgid "Starting VNC..."
msgstr "VNC indítása..."

#: vnc.py:230
msgid "The VNC server is now running."
msgstr "A VNC kiszolgáló elindult."

#: vnc.py:243
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Ön egy figyelő vncviewerhez csatlakozást választja. \n"
"Ez nem igényel beállított jelszót.  Ha beállít egy jelszót, \n"
"akkor azt használni fogom, amennyiben a csatlakozás \n"
"a vncviewerhez sikertelen\n"
"\n"

#: vnc.py:248
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"FIGYELEM!!! A VNC kiszolgáló JELSZÓ NÉLKÜL működik!\n"
"Használhatja a vncpassword=<jelszó> rendszerindító \n"
"beállítást, ha biztonságossá szeretné tenni a kiszolgálót.\n"
"\n"

#: vnc.py:252
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Ön vnc futtatását választja jelszóval. \n"
"\n"

#: vnc.py:254
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Ismeretlen hiba.  Kilépés. \n"
"\n"

#: vnc.py:276
#: vnc.py:369
msgid "VNC Configuration"
msgstr "VNC beállítása"

#: vnc.py:279
#: vnc.py:373
msgid "No password"
msgstr "Nincs jelszó"

#: vnc.py:281
#: vnc.py:376
msgid "A password will prevent unauthorized listeners connecting and monitoring your installation progress.  Please enter a password to be used for the installation"
msgstr "Jelszó használatával lehet megakadályozni, hogy illetéktelenek férjenek hozzá a telepítési folyamathoz.  Kérem adjon meg a telepítéshez használandó jelszót"

#: vnc.py:289
#: vnc.py:384
#: textw/userauth_text.py:47
msgid "Password:"
msgstr "Jelszó:"

#: vnc.py:290
#: vnc.py:385
#: textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr "Jelszó (megerősítés):"

#: vnc.py:308
#: vnc.py:407
#: textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr "Eltérő jelszavak"

#: vnc.py:309
#: vnc.py:408
#: textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr "Az ön által beírt jelszók különböznek. Kérem próbálja újra."

#: vnc.py:314
#: vnc.py:413
#: textw/userauth_text.py:66
msgid "Password Length"
msgstr "Jelszó hossz"

#: vnc.py:315
#: vnc.py:414
msgid "The password must be at least six characters long."
msgstr "A jelszó legalább 6 karakter hosszú kell legyen."

#: vnc.py:337
msgid "Unable to Start X"
msgstr "Nem lehet az X-et elindítani"

#: vnc.py:339
msgid "X was unable to start on your machine.  Would you like to start VNC to connect to this computer from another computer and perform a graphical install or continue with a text mode install?"
msgstr "Az X nem tudott elindulni az Ön gépén.  El szeretné indítani a VNC-t, hogy egy másik gépről csatlakozni lehessen ehhez a géphez - a grafikus módú telepítés indításához, vagy inkább szöveges módban szeretné folytatni a telepítést?"

#: vnc.py:358
msgid "Start VNC"
msgstr "VNC indítása"

#: vnc.py:359
#: vnc.py:361
msgid "Use text mode"
msgstr "Szöveges mód használata"

#: yuminstall.py:81
#, python-format
msgid "%s MB"
msgstr "%s MB"

#: yuminstall.py:84
#, python-format
msgid "%s KB"
msgstr "%s kB"

#: yuminstall.py:130
msgid "Preparing to install"
msgstr "Telepítés előkészítése"

#: yuminstall.py:131
msgid "Preparing transaction from installation source"
msgstr "Tranzakció előkészítése telepítő forrásból"

#: yuminstall.py:159
#, python-format
msgid "<b>Installing %s</b> (%s)\n"
msgstr "<b>%s telepítése</b> (%s)\n"

#: yuminstall.py:219
msgid "Finishing upgrade"
msgstr "Felfejlesztés befejezése"

#: yuminstall.py:220
msgid "Finishing upgrade process.  This may take a little while."
msgstr "Felfejlesztés befejezése.  Ez egy ideig eltarthat."

#: yuminstall.py:229
msgid "Error Installing Package"
msgstr "Csomag telepítése meghiúsult"

#: yuminstall.py:230
#, python-format
msgid "A fatal error occurred when installing the %s package.  This could indicate errors when reading the installation media.  Installation cannot continue."
msgstr "Végzetes hiba történt a %s csomag telepítése közben. Ez jelentheti a telepítő média hibáit is ami olvasás közben derült ki. Telepítés nem tud folytatódni."

#: yuminstall.py:300
#: iw/task_gui.py:80
#: iw/task_gui.py:319
msgid "Error Setting Up Repository"
msgstr "Hiba a tároló beállítása közben"

#: yuminstall.py:301
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%s\n"
"\n"
"Please provide the correct information for installing %s."
msgstr ""
"A következő hiba történt a telepítő tároló beállítása közben:\n"
"\n"
"%s\n"
"\n"
"Kérem adja meg a helyes információt %s telepítéséhez."

#: yuminstall.py:360
msgid "Change Disc"
msgstr "Cseréljen lemezt"

#: yuminstall.py:361
#, python-format
msgid "Please insert %s disc %d to continue."
msgstr "Helyezze be %s %d. lemezét a folytatáshoz."

#: yuminstall.py:371
msgid "Wrong Disc"
msgstr "Rossz lemez"

#: yuminstall.py:372
#, python-format
msgid "That's not the correct %s disc."
msgstr "Ez nem a megfelelő %s lemez."

#: yuminstall.py:379
msgid "Unable to access the disc."
msgstr "Elérhetetlen a lemez."

#: yuminstall.py:538
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr "%r tároló neve hiányzik a beállításban, id-t használók"

#: yuminstall.py:651
#: yuminstall.py:1065
msgid "Some of your software repositories require networking, but there was an error enabling the network on your system."
msgstr "Néhány szoftver tárolóhoz hálózat kell, de a hálózat engedélyezése közben hiba történt ."

#: yuminstall.py:712
#: yuminstall.py:714
msgid "Re_boot"
msgstr "Újrain_dítás"

#: yuminstall.py:712
msgid "_Eject"
msgstr "L_emez kiadása"

#: yuminstall.py:723
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will likely require reinstallation.\n"
"\n"
msgstr ""
"%s fájlt nem sikerült megnyitni.  Valószínűleg hiányzó fájl, sérült csomag, vagy sérült hordozó okozta.  Kérem ellenőrizze a telepítő forrást.\n"
"\n"
"Ha kilép, a rendszer töredékkel teli marad, s emiatt újbóli telepítést igényel.\n"
"\n"

#: yuminstall.py:771
msgid "Retrying"
msgstr "Újrapróbálás"

#: yuminstall.py:771
msgid "Retrying download."
msgstr "Letöltés újrapróbálása."

#: yuminstall.py:837
#, python-format
msgid "There was an error running your transaction for the following reason: %s\n"
msgstr "Az átvitel végrehajtása alatt hiba történt a következő okból: %s\n"

#: yuminstall.py:872
#: yuminstall.py:873
msgid "file conflicts"
msgstr "fájl ütközés"

#: yuminstall.py:874
msgid "older package(s)"
msgstr "régebbi csomag(ok)"

#: yuminstall.py:875
msgid "insufficient disk space"
msgstr "nem elegendő szabad lemezterület"

#: yuminstall.py:876
msgid "insufficient disk inodes"
msgstr "nincs elég inode"

#: yuminstall.py:877
msgid "package conflicts"
msgstr "csomag ütközés"

#: yuminstall.py:878
msgid "package already installed"
msgstr "már telepített csomag"

#: yuminstall.py:879
msgid "required package"
msgstr "szükséges csomag"

#: yuminstall.py:880
msgid "package for incorrect arch"
msgstr "nem megfelelő architektúrához készült csomag"

#: yuminstall.py:881
msgid "package for incorrect os"
msgstr "nem megfelelő operációs rendszerhez készült csomag"

#: yuminstall.py:895
msgid "You need more space on the following file systems:\n"
msgstr "Több szabad helyre van szüksége az alábbi fájlrendszereken:\n"

#: yuminstall.py:908
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""
"Fájlütközés történt a telepítendő csomagokat ellenőrzése alatt:\n"
"%s\n"

#: yuminstall.py:911
#, python-format
msgid "There was an error running your transaction for the following reason(s): %s.\n"
msgstr "Az átvitel végrehajtása alatt hiba történt a következő miatt: %s\n"

#: yuminstall.py:918
#: yuminstall.py:923
msgid "Error Running Transaction"
msgstr "Hiba az átvitel végrehajtása alatt"

#: yuminstall.py:1099
msgid "Unable to read group information from repositories.  This is a problem with the generation of your install tree."
msgstr "Nem sikerült beolvasni a csoportinformációt a tárolókból. A probléma a telepítési fa felépítésében lépett fel."

#: yuminstall.py:1135
msgid "Retrieving installation information."
msgstr "Telepítő információk beszerzése."

#: yuminstall.py:1137
#, python-format
msgid "Retrieving installation information for %s."
msgstr "Telepítő információk beszerzése %s számára."

#: yuminstall.py:1139
msgid "Installation Progress"
msgstr "Telepítés állapota"

#: yuminstall.py:1152
#: textw/constants_text.py:64
msgid "Edit"
msgstr "Szerkesztés"

#: yuminstall.py:1163
#: yuminstall.py:1322
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata directory.  Please ensure that your install tree has been correctly generated.\n"
"\n"
"%s"
msgstr ""
"Nem lehet beolvasni a csomagleírásokat. Ezt okozhatja a repodata mappa hiánya.  Kérem ellenőrizze, hogy jól készítették-e el a telepítő fát.\n"
"\n"
"%s"

#: yuminstall.py:1340
#, python-format
msgid "Your selected packages require %d MB of free space for installation, but you do not have enough available.  You can change your selections or exit the installer."
msgstr "A kiválasztott csomagoknak %d MB szabad hely kell, de nincs elég elérhető.  Megváltoztathatja az összeállítást, vagy kiléphet a telepítőből."

#: yuminstall.py:1359
msgid "Reboot?"
msgstr "Újraindítás?"

#: yuminstall.py:1360
msgid "The system will be rebooted now."
msgstr "A rendszer most újraindul."

#: yuminstall.py:1503
#, python-format
msgid "You appear to be upgrading from a system which is too old to upgrade to this version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr "Úgy tűnik, a rendszer túl régi, ezért nem frissíthető fel %s változatra.  Biztosan folytatni szeretné a folyamatot?"

#: yuminstall.py:1539
#, python-format
msgid "The arch of the release of %s you are upgrading to appears to be %s which does not match your previously installed arch of %s.  This is likely to not succeed.  Are you sure you wish to continue the upgrade process?"
msgstr "%s kiadás architektúrája, amire frissít, %s, ami nem egyezik meg a korábban telepített %s architektúrával.  Nem valószínű, hogy ez a folyamat sikerrel zárul.  Biztosan folytatni akarja a folyamatot?"

#: yuminstall.py:1590
msgid "Post Upgrade"
msgstr "Zárófrissítés"

#: yuminstall.py:1591
msgid "Performing post-upgrade configuration"
msgstr "Zárófrissítés beállításainak végrehajtása"

#: yuminstall.py:1593
msgid "Post Installation"
msgstr "Zárótelepítés"

#: yuminstall.py:1594
msgid "Performing post-installation configuration"
msgstr "Zárótelepítés beállításainak végrehajtása"

#: yuminstall.py:1823
msgid "Installation Starting"
msgstr "Kezdődik a telepítés"

#: yuminstall.py:1824
msgid "Starting installation process"
msgstr "Telepítési folyamat indul"

#: yuminstall.py:1862
msgid "Dependency Check"
msgstr "Függőség ellenőrzés"

#: yuminstall.py:1863
msgid "Checking dependencies in packages selected for installation"
msgstr "A telepítésre kiválasztott csomagok közötti függőségek ellenőrzése"

#: installclasses/fedora.py:39
msgid "_Fedora"
msgstr "_Fedora"

#: installclasses/fedora.py:40
#: installclasses/rhel.py:45
#, python-format
msgid "The default installation of %s includes a set of software applicable for general internet usage. What additional tasks would you like your system to include support for?"
msgstr "%s alapértelmezett telepítése tartalmaz egy általános internet használathoz készült szoftver csoportot. Milyen további feladatokat szeretne, hogy a rendszer támogasson?"

#: installclasses/fedora.py:49
msgid "Office and Productivity"
msgstr "Iroda és termelékenység"

#: installclasses/fedora.py:50
#: installclasses/rhel.py:57
#: installclasses/rhel.py:62
msgid "Software Development"
msgstr "Szoftver fejlesztés"

#: installclasses/fedora.py:51
msgid "Web Server"
msgstr "Web kiszolgáló"

#: installclasses/rhel.py:44
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: installclasses/rhel.py:54
msgid "Office"
msgstr "Iroda"

#: installclasses/rhel.py:55
msgid "Multimedia"
msgstr "Multimédia"

#: installclasses/rhel.py:61
msgid "Web server"
msgstr "Webkiszolgáló"

#: installclasses/rhel.py:66
msgid "Virtualization"
msgstr "Virtualizáció"

#: installclasses/rhel.py:67
msgid "Clustering"
msgstr "Fürtözés"

#: installclasses/rhel.py:68
msgid "Storage Clustering"
msgstr "Tároló fürtözés"

#: installclasses/rhel.py:72
msgid "Installation Number"
msgstr "Telepítés száma"

#: installclasses/rhel.py:73
msgid ""
"Would you like to enter an Installation Number (sometimes called Subscription Number) now? This feature enables the installer to access any extra components included with your subscription.  If you skip this step, additional components can be installed manually later.\n"
"\n"
"See http://www.redhat.com/InstNum/ for more information."
msgstr ""
"Szeretne megadni egy telepítés/előfizetés számot (Installation/Subscription Number)? Ez a lehetőség hozzáférést biztosít az előfizetésben foglalt extra összetevőkhöz.  Ha átugorja e lépést, később kézi vezérléssel telepíthet további összetevőket.\n"
"\n"
"Bővebb tudnivalókért lásd http://www.redhat.com/InstNum/."

#: installclasses/rhel.py:79
msgid "If you cannot locate the Installation Number, consult http://www.redhat.com/InstNum/"
msgstr "Ha nem találja a telepítés számot, tájékozódjon a http://www.redhat.com/InstNum/ lapon."

#: iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr "%s csomagok"

#: iw/GroupSelector.py:424
#, python-format
msgid "Optional packages selected: %d of %d"
msgstr "%d / %d választható csomag bejelölve."

#: iw/GroupSelector.py:425
#, python-format
msgid "<i>%s</i>"
msgstr "<i>%s</i>"

#: iw/GroupSelector.py:485
msgid "Uncategorized"
msgstr "Nem besorolt"

#: iw/account_gui.py:52
msgid "Root _Password:"
msgstr "_Rendszergazda jelszó:"

#: iw/account_gui.py:54
msgid "_Confirm:"
msgstr "_Megerősítés:"

#: iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr "Caps Lock bekapcsolva."

#: iw/account_gui.py:102
#: iw/account_gui.py:110
#: iw/account_gui.py:117
#: iw/account_gui.py:137
#: textw/userauth_text.py:74
msgid "Error with Password"
msgstr "Jelszóhiba"

#: iw/account_gui.py:103
msgid "You must enter your root password and confirm it by typing it a second time to continue."
msgstr "Meg kell adni a rendszergazda jelszót, majd megerősíteni ellenőrzésképpen, mielőtt folytatja."

#: iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr "A megadott jelszók különböznek. Kérem próbálja újra."

#: iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr "Rendszergazda jelszó legalább 6 karakter hosszú kell legyen."

#: iw/account_gui.py:127
#: textw/userauth_text.py:83
msgid "Weak Password"
msgstr "Gyenge jelszó"

#: iw/account_gui.py:128
#, python-format
msgid "You have provided a weak password: %s"
msgstr "Gyenge jelszót adott meg: %s"

#: iw/account_gui.py:130
msgid "Use Anyway"
msgstr "Mégis használjuk"

#: iw/account_gui.py:138
#: textw/userauth_text.py:75
msgid "Requested password contains non-ASCII characters, which are not allowed."
msgstr "A kért jelszó tartalmaz nem ASCII jeleket, ami nem engedélyezett."

#: iw/autopart_type.py:100
msgid "No partitions are available to resize.  Only physical partitions with specific filesystems can be resized."
msgstr "Nincs átméretezésre elérhető lemezrész.  Csak bizonyos fájlrendszerű részeket lehet átméretezni."

#: iw/autopart_type.py:123
msgid "Resize FileSystem Error"
msgstr "Fájlrendszer átméretezés hiba"

#: iw/autopart_type.py:124
#: iw/autopart_type.py:133
#, python-format
msgid "%s: %s"
msgstr "%s: %s"

#: iw/autopart_type.py:132
msgid "Resize Device Error"
msgstr "Eszköz átméretezés hiba"

#: iw/autopart_type.py:207
msgid "Do you really want to boot from a disk which is not used for installation?"
msgstr "Biztos, hogy olyan lemezről akar rendszert betölteni, amit nem használ telepítésre?"

#: iw/autopart_type.py:284
msgid "Invalid Initiator Name"
msgstr "Érvénytelen kezdeményező név"

#: iw/autopart_type.py:285
msgid "You must provide an initiator name."
msgstr "Meg kell adni egy kezdeményező nevet."

#: iw/autopart_type.py:311
msgid "Error with Data"
msgstr "Adathiba"

#: iw/autopart_type.py:469
#: iw/autopart_type.py:470
msgid "Rescanning disks"
msgstr "Lemezek lekérdezése ismét"

#: iw/autopart_type.py:524
#: textw/partition_text.py:63
msgid "Use entire drive"
msgstr "Teljes eszköz használata"

#: iw/autopart_type.py:525
#: textw/partition_text.py:64
msgid "Replace existing Linux system"
msgstr "Meglévő Linux rendszer lecserélése"

#: iw/autopart_type.py:526
msgid "Shrink current system"
msgstr "Rendszer összehúzása"

#: iw/autopart_type.py:527
#: textw/partition_text.py:65
msgid "Use free space"
msgstr "Szabad hely használata"

#: iw/autopart_type.py:528
msgid "Create custom layout"
msgstr "Egyedi felosztás teremtése"

#: iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr "Jelszó h_asználata a rendszerindítóban"

#: iw/blpasswidget.py:45
msgid "A boot loader password prevents users from changing kernel options, increasing security."
msgstr "A rendszerindító jelszavas védelme a biztonságot növelő lehetőség - megakadályozza a felhasználókat, hogy a rendszermag paramétereit elállítsák."

#: iw/blpasswidget.py:76
msgid "Change _password"
msgstr "_Jelszó megváltoztatása"

#: iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr "Adja meg a rendszerindító jelszót"

#: iw/blpasswidget.py:105
msgid "Enter a boot loader password and then confirm it.  (Note that your BIOS keymap may be different than the actual keymap you are used to.)"
msgstr "Adja meg a rendszerindító jelszót, és erősítse meg.  (Ne felejtse el, hogy a BIOS esetleg más billentyűzetkiosztást használ, mint amit ön megszokott.)"

#: iw/blpasswidget.py:112
msgid "_Password:"
msgstr "J_elszó:"

#: iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr "_Megerősítés:"

#: iw/blpasswidget.py:139
msgid "Passwords don't match"
msgstr "A jelszavak különböznek"

#: iw/blpasswidget.py:140
msgid "Passwords do not match"
msgstr "A jelszavak nem egyeznek meg"

#: iw/blpasswidget.py:149
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"A rendszerindító jelszó rövidebb 6 karakternél.  Ennél hosszabb jelszót célszerű választani.\n"
"\n"
"Folytatni szeretné ezzel a jelszóval?"

#: iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr "Rendszerindító beállítás"

#: iw/bootloader_main_gui.py:179
#: iw/bootloader_main_gui.py:184
#: iw/bootloader_main_gui.py:221
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "_Rendszerindító telepítése a /dev/%s eszközre."

#: iw/bootloader_main_gui.py:227
msgid "_Change device"
msgstr "Eszköz_csere"

#: iw/congrats_gui.py:33
msgid "Congratulations"
msgstr "Gratulálok"

#: iw/congrats_gui.py:74
#: textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""
"Gratulálok! %s telepítése kész.\n"
"\n"

#: iw/congrats_gui.py:77
#: textw/complete_text.py:42
msgid "Shutdown"
msgstr "Leállítás"

#: iw/congrats_gui.py:79
#: textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr "Kérem állítsa le a rendszert, hogy a telepített rendszert használhassa.\n"

#: iw/congrats_gui.py:81
#: textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr "Kérem indítsa újra a rendszert, hogy a telepített rendszert használhassa.\n"

#: iw/congrats_gui.py:86
#: textw/complete_text.py:51
msgid "Note that updates may be available to ensure the proper functioning of your system and installation of these updates is recommended after the reboot."
msgstr "Vegye észre, hogy a rendszer hibátlan működésének biztosításához frissítések lehetnek elérhetők, amiket ajánlunk az újraindulás után."

#: iw/congrats_gui.py:90
#: textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be available to ensure the proper functioning of your system and installation of these updates is recommended after the reboot."
msgstr ""
"Végeztünk %s telepítésével. Gratulálok!\n"
"\n"
"Kérem indítson újra, hogy a telepített rendszert használja.  Vegye észre, hogy frissítések lehetnek elérhetők a rendszer megfelelő működéséhez, és ezek telepítése ajánlott az újraindulás után."

#: iw/examine_gui.py:37
msgid "Upgrade Examine"
msgstr "Frissítési vizsgálat"

#: iw/examine_gui.py:59
#, python-format
msgid "_Install %s"
msgstr "%s tele_pítése"

#: iw/examine_gui.py:61
msgid "Choose this option to freshly install your system. Existing software and data may be overwritten depending on your configuration choices."
msgstr "Válassza ezt a lehetőséget, ha új rendszert szeretne. Az egyéb konfigurációs beállításoktól függően a meglévő szoftverek és fájlok felülírásra kerülhetnek."

#: iw/examine_gui.py:65
msgid "_Upgrade an existing installation"
msgstr "Meglévő rendszer _frissítése"

#: iw/examine_gui.py:67
#, python-format
msgid "Choose this option if you would like to upgrade your existing %s system.  This option preserves the existing data on your drives."
msgstr "Ha meglévő %s rendszert szeretne frissteni, akkor válassza ezt a lehetőséget.  Ez megtartja az eszközökön levő adatokat."

#: iw/examine_gui.py:123
msgid "The following installed system will be upgraded:"
msgstr "A következő telepített rendszert fogom frissíteni:"

#: iw/examine_gui.py:136
msgid "Unknown Linux system"
msgstr "Ismeretlen Linux rendszer"

#: iw/language_gui.py:33
msgid "Language Selection"
msgstr "Nyelv kiválasztása"

#: iw/language_gui.py:81
#: loader/lang.c:377
msgid "What language would you like to use during the installation process?"
msgstr "Melyik nyelvet szeretné használni a telepítés alatt?"

#: iw/lvm_dialog_gui.py:121
#: iw/lvm_dialog_gui.py:167
#: iw/lvm_dialog_gui.py:179
#: iw/lvm_dialog_gui.py:218
#: iw/lvm_dialog_gui.py:295
#: iw/lvm_dialog_gui.py:662
#: iw/lvm_dialog_gui.py:682
msgid "Not enough space"
msgstr "Nincs elég szabad hely"

#: iw/lvm_dialog_gui.py:122
msgid "The physical extent size cannot be changed because otherwise the space required by the currently defined logical volumes will be increased to more than the available space."
msgstr "A fizikai terjedelem mérete nem változtatható meg, mert akkor a már meglévő logikai kötetek mérete meghaladná a rendelkezésre álló helyet."

#: iw/lvm_dialog_gui.py:131
msgid "Confirm Physical Extent Change"
msgstr "Fizikai terjedelem méretének változtatás megerősítése"

#: iw/lvm_dialog_gui.py:132
msgid ""
"This change in the value of the physical extent will require the sizes of the current logical volume requests to be rounded up in size to an integer multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""
"Fizikai terjedelem értékének megváltozása megköveteli, hogy a már meglevő logikai kötet kérelmek mérete a fizikai kiterjedés méretének egész számú többszörösére legyen felkerekítve.\n"
"\n"
"A változás azonnal érvénybe lép."

#: iw/lvm_dialog_gui.py:141
#: iw/lvm_dialog_gui.py:201
msgid "C_ontinue"
msgstr "F_olytatás"

#: iw/lvm_dialog_gui.py:168
#, python-format
msgid "The physical extent size cannot be changed because the value selected (%10.2f MB) is larger than the smallest physical volume (%10.2f MB) in the volume group."
msgstr "Fizikai kiterjedés mérete nem változtatható meg, mert a választott érték (%10.2f MB) nagyobb, mint a legkisebb fizikai kötet (%10.2f MB) a kötet csoportban."

#: iw/lvm_dialog_gui.py:180
#, python-format
msgid "The physical extent size cannot be changed because the value selected (%10.2f MB) is too large compared to the size of the smallest physical volume (%10.2f MB) in the volume group."
msgstr "Fizikai kiterjedés mérete nem változtatható meg, mert a választott érték (%10.2f MB) túl nagy a legkisebb fizikai kötethez (%10.2f MB) viszonyítva a kötetcsoportban."

#: iw/lvm_dialog_gui.py:194
msgid "Too small"
msgstr "Túl kicsi"

#: iw/lvm_dialog_gui.py:195
msgid "This change in the value of the physical extent will waste substantial space on one or more of the physical volumes in the volume group."
msgstr "Ha erre az értékre változtatja a fizikai kiterjedés méretét, akkor a kötet csoporton belül legalább egy fizikai köteten jelentős helyet pocsékol el."

#: iw/lvm_dialog_gui.py:219
#, python-format
msgid "The physical extent size cannot be changed because the resulting maximum logical volume size (%10.2f MB) is smaller than one or more of the currently defined logical volumes."
msgstr "Fizikai kiterjedés mérete nem változtatható meg, mert ezzel a maximális logikai kötet méret (%10.2f MB) kisebb lenne, mint az egyik létező logikai kötet mérete."

#: iw/lvm_dialog_gui.py:296
msgid "You cannot remove this physical volume because otherwise the volume group will be too small to hold the currently defined logical volumes."
msgstr "E fizikai kötet nem távolítható el, mert akkor a logikai kötetcsoport túl kicsi lenne ahhoz, hogy elférjenek rajta a most megadott logikai kötetek."

#: iw/lvm_dialog_gui.py:395
msgid "Make Logical Volume"
msgstr "Logikai kötet készítése"

#: iw/lvm_dialog_gui.py:397
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "Logikai kötet szerkesztése: %s"

#: iw/lvm_dialog_gui.py:428
#: iw/raid_dialog_gui.py:380
msgid "_File System Type:"
msgstr "_Fájl rendszer típusa:"

#: iw/lvm_dialog_gui.py:434
msgid "_Logical Volume Name:"
msgstr "_Logikai kötet név:"

#: iw/lvm_dialog_gui.py:443
#: iw/partition_dialog_gui.py:417
msgid "_Size (MB):"
msgstr "Mé_ret (MB):"

#: iw/lvm_dialog_gui.py:449
#, python-format
msgid "(Max size is %s MB)"
msgstr "(Max. méret %s MB)"

#: iw/lvm_dialog_gui.py:453
#: iw/partition_dialog_gui.py:469
#: iw/partition_ui_helpers_gui.py:329
#: iw/raid_dialog_gui.py:352
msgid "_Encrypt"
msgstr "_Titkosítás"

#: iw/lvm_dialog_gui.py:462
#: iw/raid_dialog_gui.py:390
msgid "Original File System Type:"
msgstr "Eredeti fájl rendszer típus:"

#: iw/lvm_dialog_gui.py:466
#: iw/partition_gui.py:776
#: iw/partition_gui.py:928
#: iw/raid_dialog_gui.py:395
msgid "Unknown"
msgstr "Ismeretlen"

#: iw/lvm_dialog_gui.py:470
#: iw/partition_dialog_gui.py:407
#: iw/raid_dialog_gui.py:401
msgid "Original File System Label:"
msgstr "Eredeti fájl rendszer címke:"

#: iw/lvm_dialog_gui.py:475
msgid "Logical Volume Name:"
msgstr "Logikai kötet név:"

#: iw/lvm_dialog_gui.py:479
msgid "Size (MB):"
msgstr "Méret (MB):"

#: iw/lvm_dialog_gui.py:506
#: iw/partition_dialog_gui.py:362
#: iw/raid_dialog_gui.py:371
msgid "_Mount Point:"
msgstr "_Csatolási pont:"

#: iw/lvm_dialog_gui.py:589
msgid "Illegal Logical Volume Name"
msgstr "Tiltott logikai kötet név"

#: iw/lvm_dialog_gui.py:604
msgid "Illegal logical volume name"
msgstr "Tiltott logikai kötet név"

#: iw/lvm_dialog_gui.py:605
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr "Ezt a „%s” logikai kötet nevet már használják. Kérem válasszon másikat."

#: iw/lvm_dialog_gui.py:633
msgid "Mount point in use"
msgstr "Használt csatoló pont"

#: iw/lvm_dialog_gui.py:634
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr "Ezt a „%s” csatoló pontot már használják. Kérem válasszon másikat."

#: iw/lvm_dialog_gui.py:649
msgid "Illegal size"
msgstr "Tiltott méret"

#: iw/lvm_dialog_gui.py:650
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "A megadott méret érvényes pozitív szám kell lengyen."

#: iw/lvm_dialog_gui.py:663
#, python-format
msgid "The current requested size (%10.2f MB) is larger than the maximum logical volume size (%10.2f MB). To increase this limit you can create more Physical Volumes from unpartitioned disk space and add them to this Volume Group."
msgstr "A kért méret (%10.2f MB) nagyobb, mint a maximális logikai kötet méret (%10.2f MB). Ha szeretné növelni ez utóbbi korlátot, akkor csináljon további fizikai kötetet a még fel nem osztott helyen, és adja azt hozzá e kötetcsoporthoz."

#: iw/lvm_dialog_gui.py:683
#, python-format
msgid "The logical volumes you have configured require %d MB, but the volume group only has %d MB.  Please either make the volume group larger or make the logical volume(s) smaller."
msgstr "A beállított logikai köteteknek %d MB kell, de a kötetcsoportban csak %d MB van.  Kérem bővítse a kötetcsoportot, vagy szűkítse a logikai kötetet."

#: iw/lvm_dialog_gui.py:770
msgid "No free slots"
msgstr "Nincs üres aljzat"

#: iw/lvm_dialog_gui.py:781
msgid "No free space"
msgstr "Nincs szabad hely"

#: iw/lvm_dialog_gui.py:782
msgid "There is no room left in the volume group to create new logical volumes. To add a logical volume you must reduce the size of one or more of the currently existing logical volumes"
msgstr "Nem maradt hely a kötetcsoportban új logikai kötet létrehozásához. Logikai kötet hozzáadásához legalább az egyik meglévő logikai kötet méretét le kell csökkenteni."

#: iw/lvm_dialog_gui.py:814
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "Biztosan törölni szeretné „%s” logikai kötetet?"

#: iw/lvm_dialog_gui.py:927
msgid "Invalid Volume Group Name"
msgstr "Érvénytelen kötet csoport név"

#: iw/lvm_dialog_gui.py:936
msgid "Name in use"
msgstr "Név használatban"

#: iw/lvm_dialog_gui.py:937
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr "„%s” kötet csoport nevet már használják. Kérem válasszon másikat."

#: iw/lvm_dialog_gui.py:1180
msgid "Not enough physical volumes"
msgstr "Nincs elég fizikai kötet"

#: iw/lvm_dialog_gui.py:1181
msgid ""
"At least one unused physical volume partition is needed to create an LVM Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then select the \"LVM\" option again."
msgstr ""
"Legalább egy nem használt fizikai kötet rész szükséges LVM kötetcsoport teremtéséhez.\n"
"\n"
"Teremtsen egy „fizikai kötet (LVM)” típusú részt vagy egy RAID csoportot majd válassza újból az „LVM” opciót."

#: iw/lvm_dialog_gui.py:1192
msgid "Make LVM Volume Group"
msgstr "LVM kötet csoport készítése"

#: iw/lvm_dialog_gui.py:1195
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "LVM kötet csoport szerkesztése: %s"

#: iw/lvm_dialog_gui.py:1197
msgid "Edit LVM Volume Group"
msgstr "LVM kötet csoport szerkesztése"

#: iw/lvm_dialog_gui.py:1213
msgid "_Volume Group Name:"
msgstr "Kötet cso_port név:"

#: iw/lvm_dialog_gui.py:1221
msgid "Volume Group Name:"
msgstr "Kötet csoport név:"

#: iw/lvm_dialog_gui.py:1229
msgid "_Physical Extent:"
msgstr "_Fizikai darab:"

#: iw/lvm_dialog_gui.py:1244
msgid "Physical Volumes to _Use:"
msgstr "_Használandó fizikai kötetek:"

#: iw/lvm_dialog_gui.py:1250
msgid "Used Space:"
msgstr "Felhasznált hely:"

#: iw/lvm_dialog_gui.py:1267
msgid "Free Space:"
msgstr "Szabad hely:"

#: iw/lvm_dialog_gui.py:1285
msgid "Total Space:"
msgstr "Összes hely:"

#: iw/lvm_dialog_gui.py:1323
msgid "Logical Volume Name"
msgstr "Logikai kötet név"

#: iw/lvm_dialog_gui.py:1326
#: iw/partition_gui.py:362
#: iw/upgrade_swap_gui.py:139
#: textw/upgrade_text.py:118
msgid "Mount Point"
msgstr "Csatoló pont"

#: iw/lvm_dialog_gui.py:1329
#: iw/partition_gui.py:367
msgid "Size (MB)"
msgstr "Méret (MB)"

#: iw/lvm_dialog_gui.py:1343
#: iw/osbootwidget.py:95
msgid "_Add"
msgstr "_Hozzáadás"

#: iw/lvm_dialog_gui.py:1346
#: iw/osbootwidget.py:99
#: iw/partition_gui.py:1336
msgid "_Edit"
msgstr "Sz_erkesztés"

#: iw/lvm_dialog_gui.py:1361
msgid "_Logical Volumes"
msgstr "_Logikai kötetek"

#: iw/netconfig_dialog.py:192
#: textw/netconfig_text.py:36
#, python-format
msgid ""
"An error occurred converting the value entered for \"%s\":\n"
"%s"
msgstr ""
"„%s” mezőben megadott adatot átalakítván hiba történt:\n"
"%s"

#: iw/netconfig_dialog.py:194
#: iw/netconfig_dialog.py:203
#: textw/netconfig_text.py:35
#: textw/netconfig_text.py:40
msgid "Error With Data"
msgstr "Hibás adat"

#: iw/netconfig_dialog.py:202
#, python-format
msgid "A value is required for the field %s."
msgstr "Meg kell adni egy értéket „%s” mezőnek."

#: iw/netconfig_dialog.py:212
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr "%s hálózati csatolót bekapcsolván hiba történt."

#: iw/netconfig_dialog.py:214
msgid "Error Configuring Network"
msgstr "Hálózat beállítás hiba"

#: iw/netconfig_dialog.py:240
msgid "Dynamic IP Address"
msgstr "Változó IP cím"

#: iw/netconfig_dialog.py:241
#, python-format
msgid "Sending request for IP address information for %s"
msgstr "%s IP adatainak lekérdezése"

#: iw/netconfig_dialog.py:256
#: iw/netconfig_dialog.py:259
#: textw/netconfig_text.py:223
#: textw/netconfig_text.py:226
msgid "IP Address"
msgstr "IP cím"

#: iw/netconfig_dialog.py:266
#: textw/netconfig_text.py:233
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "IPv4 CIDR előtag 0 és 32 közé kell essen."

#: iw/netconfig_dialog.py:267
#: iw/netconfig_dialog.py:273
#: iw/netconfig_dialog.py:281
#: iw/netconfig_dialog.py:284
#: textw/netconfig_text.py:234
#: textw/netconfig_text.py:240
#: textw/netconfig_text.py:248
msgid "IPv4 Network Mask"
msgstr "IPv4 hálózati maszk"

#: iw/netconfig_dialog.py:294
#: textw/netconfig_text.py:261
msgid "Gateway"
msgstr "Átjáró"

#: iw/netconfig_dialog.py:304
#: textw/netconfig_text.py:271
msgid "Nameserver"
msgstr "Névkiszolgáló"

#: iw/netconfig_dialog.py:313
msgid "Error configuring network device:"
msgstr "Hiba a hálózati eszköz beállítása közben:"

#: iw/network_gui.py:67
#: iw/network_gui.py:73
msgid "Error with Hostname"
msgstr "Hibás gazda név"

#: iw/network_gui.py:68
msgid "You must enter a valid hostname for this computer."
msgstr "Érvényes gazda nevet kell adni ennek a számítógépnek."

#: iw/network_gui.py:74
#, python-format
msgid ""
"The hostname \"%s\" is not valid for the following reason:\n"
"\n"
"%s"
msgstr ""
"A megadott „%s” gazda név érvénytelen az alábbi ok miatt:\n"
"\n"
"%s"

#: iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr "Rendszerindító operációs rendszer lista"

#: iw/osbootwidget.py:66
msgid "Default"
msgstr "Alapértelmezés"

#: iw/osbootwidget.py:66
#: iw/partition_gui.py:361
msgid "Label"
msgstr "Címke"

#: iw/osbootwidget.py:66
#: iw/partition_gui.py:360
msgid "Device"
msgstr "Eszköz"

#: iw/osbootwidget.py:129
msgid "Image"
msgstr "Kép"

#: iw/osbootwidget.py:136
msgid "Enter a label for the boot loader menu to display. The device (or hard drive and partition number) is the device from which it boots."
msgstr "Adja meg a rendszerindító menüben megjelenítendő címkét. A megadott eszközről (vagy merevlemez és partíció számról) indul majd a rendszer."

#: iw/osbootwidget.py:144
msgid "_Label"
msgstr "_Címke"

#: iw/osbootwidget.py:152
msgid "_Device"
msgstr "Esz_köz"

#: iw/osbootwidget.py:182
msgid "Default Boot _Target"
msgstr "Alapértelme_zett rendszerindító cél"

#: iw/osbootwidget.py:211
msgid "You must specify a label for the entry"
msgstr "Meg kell adni egy címkét a bejegyzéshez"

#: iw/osbootwidget.py:220
msgid "Boot label contains illegal characters"
msgstr "A rendszerindító címke tiltott karaktereket tartalmaz"

#: iw/osbootwidget.py:244
msgid "Duplicate Label"
msgstr "Ismételt címke"

#: iw/osbootwidget.py:245
msgid "This label is already in use for another boot entry."
msgstr "Ezt a címkét egy másik rendszerindító bejegyzés már használja."

#: iw/osbootwidget.py:258
msgid "Duplicate Device"
msgstr "Ismételt eszköz"

#: iw/osbootwidget.py:259
msgid "This device is already being used for another boot entry."
msgstr "Ezt az eszközt egy másik rendszerindító bejegyzés már használja."

#: iw/osbootwidget.py:321
msgid "Cannot Delete"
msgstr "Nem lehet törölni"

#: iw/osbootwidget.py:322
#, python-format
msgid "This boot target cannot be deleted because it is for the %s system you are about to install."
msgstr "Ez a rendszerindító bejegyzés nem törölhető, mert a most telepítendő %s rendszerhez tartozik."

#: iw/partition_dialog_gui.py:57
msgid "Additional Size Options"
msgstr "További méret lehetőségek"

#: iw/partition_dialog_gui.py:62
msgid "_Fixed size"
msgstr "_Rögzített méret"

#: iw/partition_dialog_gui.py:64
msgid "Fill all space _up to (MB):"
msgstr "Összes hely kitöltése _eddig (MB):"

#: iw/partition_dialog_gui.py:74
msgid "Fill to maximum _allowable size"
msgstr "Kitöltés a max. _megengedett méretig"

#: iw/partition_dialog_gui.py:332
msgid "Add Partition"
msgstr "Partició hozzáadása"

#: iw/partition_dialog_gui.py:334
#, python-format
msgid "Edit Partition: %s"
msgstr "%s partíció szerkesztése"

#: iw/partition_dialog_gui.py:371
msgid "File System _Type:"
msgstr "Fájl rendszer _típusa:"

#: iw/partition_dialog_gui.py:387
msgid "Allowable _Drives:"
msgstr "Engedélyezett e_szközök:"

#: iw/partition_dialog_gui.py:456
msgid "Force to be a _primary partition"
msgstr "Kényszerített első_dleges rész legyen"

#: iw/partition_gui.py:319
#, python-format
msgid "Drive %s (%-0.f MB) (Model: %s)"
msgstr "%s eszköz (%-0.f MB) (modell: %s)"

#: iw/partition_gui.py:363
msgid "Type"
msgstr "Típus"

#: iw/partition_gui.py:366
#: storage/__init__.py:1630
msgid "Format"
msgstr "Formázás"

#: iw/partition_gui.py:404
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""
"Csatolási pont/\n"
"RAID/Kötet"

#: iw/partition_gui.py:406
msgid ""
"Size\n"
"(MB)"
msgstr ""
"Méret\n"
"(MB)"

#: iw/partition_gui.py:518
msgid "Partitioning"
msgstr "Partícionálás"

#: iw/partition_gui.py:608
msgid "The partitioning scheme you requested caused the following critical errors."
msgstr "A kért felosztási séma a következő kritikus hibákat okozta:"

#: iw/partition_gui.py:610
#, python-format
msgid "You must correct these errors before you continue your installation of %s."
msgstr "E hibákat ki kell javítani, mielőtt folytatná %s telepítését."

#: iw/partition_gui.py:616
msgid "Partitioning Errors"
msgstr "Partícionálási hibák"

#: iw/partition_gui.py:623
msgid "The partitioning scheme you requested generated the following warnings."
msgstr "A kért felosztási séma a következő figyelmeztetéseket idézte elő:"

#: iw/partition_gui.py:625
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr "Tovább szeretne lépni a kért felosztási sémával?"

#: iw/partition_gui.py:630
msgid "Partitioning Warnings"
msgstr "Partícionálási figyelmeztetések"

#: iw/partition_gui.py:651
msgid "Format Warnings"
msgstr "Formázási figyelmeztetések"

#: iw/partition_gui.py:656
msgid "_Format"
msgstr "_Formázás"

#: iw/partition_gui.py:683
msgid "LVM Volume Groups"
msgstr "LVM kötet csoportok"

#: iw/partition_gui.py:730
msgid "RAID Devices"
msgstr "RAID eszközök"

#: iw/partition_gui.py:797
#: loader/hdinstall.c:222
msgid "Hard Drives"
msgstr "Merevlemezek"

#: iw/partition_gui.py:896
msgid "Extended"
msgstr "Kibővített"

#: iw/partition_gui.py:898
msgid "software RAID"
msgstr "szoftver RAID"

#: iw/partition_gui.py:930
msgid "Free"
msgstr "Szabad"

#: iw/partition_gui.py:1023
#: storage/partitioning.py:190
#: storage/partitioning.py:231
msgid "Error Partitioning"
msgstr "Hiba a particionáláskor"

#: iw/partition_gui.py:1024
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "Nem sikerült lefoglalni a kért partíciókat: %s."

#: iw/partition_gui.py:1033
#, python-format
msgid "Warning: %s."
msgstr "Figyelmeztetés: %s."

#: iw/partition_gui.py:1063
#: iw/partition_gui.py:1070
msgid "Unable To Edit"
msgstr "Nem lehet szerkeszteni"

#: iw/partition_gui.py:1064
msgid "You must select a device to edit"
msgstr "Ki kell választani a szerkeszteni kívánt eszközt"

#: iw/partition_gui.py:1071
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"Ezen rész nem szerkeszthető:\n"
"\n"
"%s"

#: iw/partition_gui.py:1178
#: iw/partition_gui.py:1190
msgid "Not supported"
msgstr "Nem támogatott"

#: iw/partition_gui.py:1179
msgid "LVM is NOT supported on this platform."
msgstr "LVM nem támogatott ezen a platformon."

#: iw/partition_gui.py:1191
msgid "Software RAID is NOT supported on this platform."
msgstr "Szoftveres RAID nem támogatott ezen a platformon."

#: iw/partition_gui.py:1198
msgid "No RAID minor device numbers available"
msgstr "Nem áll rendelkezésre kis eszköz szám (minor device number)."

#: iw/partition_gui.py:1199
msgid "A software RAID device cannot be created because all of the available RAID minor device numbers have been used."
msgstr "Nem lehet szoftver RAID eszközt készíteni, mert az összes felhasználható kis eszköz szám már foglalt."

#: iw/partition_gui.py:1211
msgid "RAID Options"
msgstr "RAID opciók"

#: iw/partition_gui.py:1226
#, python-format
msgid "Software RAID allows you to combine several disks into a larger RAID device.  A RAID device can be configured to provide additional speed and reliability compared to using an individual drive.  For more information on using RAID devices please consult the %s documentation."
msgstr "Szoftveres RAID esetén több lemezt össze lehet vonni egy nagyobb RAID eszközbe.  RAID eszközök külön álló lemezekhez képest nagyobb sebességet, és megbízhatóságot nyújtanak.  RAID eszközök használatával kapcsolatban kérem olvassa el a %s dokumentációt."

#: iw/partition_gui.py:1236
msgid ""
"To use RAID you must first create at least two partitions of type 'software RAID'.  Then you can create a RAID device that can be formatted and mounted.\n"
"\n"
msgstr ""
"RAID használatához először készíteni kell legalább két 'szoftveres RAID' típusú részt.  Majd egy RAID eszközt, ami kialakítható és becsatolható.\n"
"\n"

#: iw/partition_gui.py:1242
msgid "What do you want to do now?"
msgstr "Most mit szeretne tenni?"

#: iw/partition_gui.py:1251
msgid "Create a software RAID _partition."
msgstr "Szoftveres RAID _részt szeretnék készíteni."

#: iw/partition_gui.py:1254
#, python-format
msgid "Create a RAID _device [default=/dev/md%s]."
msgstr "RAI_D tömböt szeretnék készíteni [alapértelmezés=/dev/md%s]."

#: iw/partition_gui.py:1258
#, python-format
msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
msgstr "Tükrözni szeretnék egy e_szközt, hogy RAID tömböt készítsek [alapértelmezés=/dev/md%s]."

#: iw/partition_gui.py:1300
msgid "Couldn't Create Drive Clone Editor"
msgstr "Meghajtó Tükröző létrehozása sikertelen"

#: iw/partition_gui.py:1301
msgid "The drive clone editor could not be created for some reason."
msgstr "Valamilyen ok miatt nem sikerült Meghajtó Tükröző-t létrehozni."

#: iw/partition_gui.py:1335
msgid "Ne_w"
msgstr "Ú_j"

#: iw/partition_gui.py:1338
msgid "Re_set"
msgstr "_Visszaállítás"

#: iw/partition_gui.py:1339
msgid "R_AID"
msgstr "R_AID"

#: iw/partition_gui.py:1340
msgid "_LVM"
msgstr "_LVM"

#: iw/partition_gui.py:1381
msgid "Hide RAID device/LVM Volume _Group members"
msgstr "RAID eszköz/LVM kötet csoport tagok elrejtése"

#: iw/partition_ui_helpers_gui.py:108
#: iw/partition_ui_helpers_gui.py:130
#: iw/partition_ui_helpers_gui.py:132
msgid "<Not Applicable>"
msgstr "<Nem jellemző>"

#: iw/partition_ui_helpers_gui.py:310
msgid "_Format as:"
msgstr "_Formázás mint:"

#: iw/partition_ui_helpers_gui.py:332
msgid "Mi_grate filesystem to:"
msgstr "Fájlrendszer mi_grálása:"

#: iw/partition_ui_helpers_gui.py:362
msgid "_Resize"
msgstr "Át_méretezés"

#: iw/partition_ui_helpers_gui.py:431
#, python-format
msgid "Partitions of type '%s' must be constrained to a single drive.  To do this, select the drive in the 'Allowable Drives' checklist."
msgstr "„%s” típusú részeket csak egyetlen eszközhöz lehet rendelni.  Az eszközt a „Használható eszközök” listából lehet kiválasztani."

#: iw/progress_gui.py:37
msgid "Installing Packages"
msgstr "Csomagok telepítése"

#: iw/raid_dialog_gui.py:319
msgid ""
"At least two unused software RAID partitions are needed to create a RAID device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then select the \"RAID\" option again."
msgstr ""
"Legalább két szoftveres RAID rész szükséges RAID eszköz készítéséhez.\n"
"\n"
"Először csináljon legalább két „szoftver RAID” részt, majd válassza újból a „RAID” opciót."

#: iw/raid_dialog_gui.py:333
#: iw/raid_dialog_gui.py:770
msgid "Make RAID Device"
msgstr "RAID eszköz készítése"

#: iw/raid_dialog_gui.py:336
#, python-format
msgid "Edit RAID Device: %s"
msgstr "%s RAID eszköz szerkesztése"

#: iw/raid_dialog_gui.py:338
msgid "Edit RAID Device"
msgstr "RAID eszköz szerkesztése"

#: iw/raid_dialog_gui.py:409
msgid "RAID _Device:"
msgstr "RAID _eszköz:"

#: iw/raid_dialog_gui.py:427
msgid "RAID _Level:"
msgstr "RAID _szint:"

#: iw/raid_dialog_gui.py:469
msgid "_RAID Members:"
msgstr "_RAID tagok:"

#: iw/raid_dialog_gui.py:486
msgid "Number of _spares:"
msgstr "Tartalékok _száma:"

#: iw/raid_dialog_gui.py:496
msgid "_Format partition?"
msgstr "Partíció _formázása?"

#: iw/raid_dialog_gui.py:570
msgid "The source drive has no partitions to be cloned.  You must first define partitions of type 'software RAID' on this drive before it can be cloned."
msgstr "A forrás eszközön nincs tükrözhető rész.  Másolás előtt „szoftveres RAID” típusú részeket kell készíteni az eszközön."

#: iw/raid_dialog_gui.py:574
#: iw/raid_dialog_gui.py:580
#: iw/raid_dialog_gui.py:594
#: iw/raid_dialog_gui.py:607
msgid "Source Drive Error"
msgstr "Hibás forrás eszköz"

#: iw/raid_dialog_gui.py:581
msgid ""
"The source drive you selected has partitions which are not of type 'software RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""
"A kijelölt forrás eszközön meglévő részek között nem „szoftveres RAID” típusúak is vannak.\n"
"\n"
"Az eszköz tükrözése csak e részek eltávolítása után lehetséges. "

#: iw/raid_dialog_gui.py:595
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to the drive %s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this drive can be cloned. "
msgstr ""
"A kijelölt forrás eszközön van olyan rész, ami nem csak %s eszközre korlátozódik.\n"
"\n"
"Ezeket a partíciókat el kell távolítani, vagy erre az eszközre kell korlátozni ahhoz, hogy az eszköz tükrözhető legyen. "

#: iw/raid_dialog_gui.py:608
msgid ""
"The source drive you selected has software RAID partition(s) which are members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""
"A kijelölt forrás eszköz tartalmaz szoftver RAID részeket, amik egy aktív szoftveres RAID eszköz tagjai.\n"
"\n"
"E részeket el kell távolítani ahhoz, hogy az eszköz másolható legyen."

#: iw/raid_dialog_gui.py:622
#: iw/raid_dialog_gui.py:628
#: iw/raid_dialog_gui.py:644
msgid "Target Drive Error"
msgstr "Hibás cél eszköz"

#: iw/raid_dialog_gui.py:623
msgid "Please select the target drives for the clone operation."
msgstr "Válasszon cél eszközöket a tükrözési művelethez."

#: iw/raid_dialog_gui.py:629
#, python-format
msgid "The source drive %s cannot be selected as a target drive as well."
msgstr "%s forrás eszköz nem választható ki cél eszköznek is egyszerre."

#: iw/raid_dialog_gui.py:645
#, python-format
msgid ""
"The target drive %s has a partition which cannot be removed for the following reason:\n"
"\n"
"\"%s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""
"A %s cél eszköz tartalmaz olyan részt, ami a következő okból nem távolítható el:\n"
"\n"
"„%s”\n"
"\n"
"Csak e rész eltávolítása után jelölhető ki célként az eszköz."

#: iw/raid_dialog_gui.py:708
msgid "Please select a source drive."
msgstr "Válasszon forrás meghajtót."

#: iw/raid_dialog_gui.py:728
#, python-format
msgid ""
"The drive %s will now be cloned to the following drives:\n"
"\n"
msgstr ""
"A %s eszközt a következő eszközökre tükrözöm:\n"
"\n"

#: iw/raid_dialog_gui.py:733
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""
"\n"
"\n"
"FIGYELEM! A CÉL ESZKÖZÖKÖN MEGLÉVŐ ADATOK ELVESZNEK!"

#: iw/raid_dialog_gui.py:736
msgid "Final Warning"
msgstr "Utolsó figyelmeztetés"

#: iw/raid_dialog_gui.py:738
msgid "Clone Drives"
msgstr "Eszközök tükrözése"

#: iw/raid_dialog_gui.py:747
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr "Hiba történt a cél eszközök törlése közben.  A tükrözés sikertelen."

#: iw/raid_dialog_gui.py:780
msgid ""
"Clone Drive Tool\n"
"\n"
"This tool allows you to significantly reduce the amount of effort required to setup RAID arrays.  This tool uses a source drive which has been prepared with the desired partitioning layout, and clones this layout onto other similar sized drives.  Then a RAID device can be created.\n"
"\n"
"NOTE: The source drive must have partitions which are restricted to be on that drive only, and can only contain unused software RAID partitions.  Other partition types are not allowed.\n"
"\n"
"EVERYTHING on the target drive(s) will be destroyed by this process."
msgstr ""
"Meghajtó Tükröző alkalmazás\n"
"\n"
"Ez az alkalmazás jelentősen csökkenti a RAID tömbök készítésére, illetve beállítására fordítandó erőfeszítéseket.  Ez az eszköz egy forrás meghajtót használ, amin a kívánt partícionálást kialakították, és ezt a felosztást tükrözi más hasonló méretű tároló eszközökre.  Ezek után lehet RAID eszközt elkészíteni.\n"
"\n"
"Megjegyzés: A forrás meghajtó részei nem nyúlhatnak át más eszközökre, továbbá az eszköz csak felhasználatlan, szoftveres RAID típusú részeket tartalmazhat. Más partíció típusok nem engedélyezettek.\n"
"\n"
"A folyamat során a cél eszköz(ök)ön meglévő MINDEN adat elveszik."

#: iw/raid_dialog_gui.py:800
msgid "Source Drive:"
msgstr "Forrás eszköz:"

#: iw/raid_dialog_gui.py:808
msgid "Target Drive(s):"
msgstr "Cél eszköz(ök):"

#: iw/raid_dialog_gui.py:816
msgid "Drives"
msgstr "Eszközök"

#: iw/task_gui.py:64
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a missing repodata directory.  Please ensure that your repository has been correctly generated.\n"
"\n"
"%s"
msgstr ""
"Nem lehet beolvasni a csomagleíró adatokat a tárolóból.  Talán hiányzik egy repodata mappa.  Kérem ellenőrizze, hogy jól készítették-e el a tárolót.\n"
"\n"
"%s"

#: iw/task_gui.py:81
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%s\n"
"\n"
"Please provide the correct information for installing %s"
msgstr ""
"A következő hiba történt a telepítő tárolót beállításakor:\n"
"\n"
"%s\n"
"\n"
"Kérem adja meg a helyes információt %s telepítéséhez."

#: iw/task_gui.py:154
msgid "Edit Repository"
msgstr "Tároló szerkesztése"

#: iw/task_gui.py:179
#, python-format
msgid "The repository %s has already been added.  Please choose a different repository name and URL."
msgstr "„%s” tárolót már hozzáadta. Kérem válasszon másik tároló nevet és URL-t."

#: iw/task_gui.py:254
msgid "Invalid Proxy URL"
msgstr "Érvénytelen proxy URL"

#: iw/task_gui.py:255
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr "Meg kell adnia egy HTTP, HTTPS vagy FTP URL-t egy proxyhoz."

#: iw/task_gui.py:266
#: iw/task_gui.py:404
msgid "Invalid Repository URL"
msgstr "Érvénytelen tároló URL"

#: iw/task_gui.py:267
#: iw/task_gui.py:405
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr "Meg kell adnia egy HTTP, HTTPS vagy FTP URL-t egy tárolóhoz."

#: iw/task_gui.py:288
#: iw/task_gui.py:414
msgid "No Media Found"
msgstr "Nem találtam hordozót"

#: iw/task_gui.py:289
#: iw/task_gui.py:415
msgid "No installation media was found. Please insert a disc into your drive and try again."
msgstr "Nem találtam telepítő hordozót. Kérem tegyen be egy lemezt, és próbálja újra."

#: iw/task_gui.py:307
#: iw/task_gui.py:431
msgid "Please enter an NFS server and path."
msgstr "Kérem adjon meg NFS kiszolgáló nevet és utat."

#: iw/task_gui.py:320
#, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""
"A következő hiba merült fel a tároló beállítása közben:\n"
"\n"
"%s"

#: iw/task_gui.py:342
msgid "Invalid Repository Name"
msgstr "Érvénytelen tároló név"

#: iw/task_gui.py:343
msgid "You must provide a repository name."
msgstr "Meg kell adnia egy tároló nevet."

#: iw/task_gui.py:463
#: ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr "Tároló hozzáadása"

#: iw/task_gui.py:468
msgid "No Software Repos Enabled"
msgstr "Nincs engedélyezett szoftver tároló"

#: iw/task_gui.py:469
msgid "You must have at least one software repository enabled to continue installation."
msgstr "Legalább egy szoftver tárolót engedélyezni kell, hogy folytassuk a telepítést."

#: iw/timezone_gui.py:63
#: textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr "Időzóna választás"

#: iw/upgrade_bootloader_gui.py:37
#: textw/upgrade_bootloader_text.py:136
msgid "Upgrade Boot Loader Configuration"
msgstr "Rendszerbetöltő beállítások frissítése"

#: iw/upgrade_bootloader_gui.py:123
msgid "_Update boot loader configuration"
msgstr "Rendszerbetöltő beállításainak _frissítése"

#: iw/upgrade_bootloader_gui.py:124
msgid "This will update your current boot loader."
msgstr "Ez frissíti az aktuális rendszerbetöltőt."

#: iw/upgrade_bootloader_gui.py:128
#: textw/upgrade_bootloader_text.py:108
msgid "Due to system changes, your boot loader configuration can not be automatically updated."
msgstr "A rendszerben történt módosítások miatt a rendszerindító beállításai nem frissíthetők automatikusan."

#: iw/upgrade_bootloader_gui.py:131
#: textw/upgrade_bootloader_text.py:112
msgid "The installer is unable to detect the boot loader currently in use on your system."
msgstr "A telepítő nem képes érzékelni a most használt rendszerindítót."

#: iw/upgrade_bootloader_gui.py:138
#: textw/upgrade_bootloader_text.py:121
#, python-format
msgid "The installer has detected the %s boot loader currently installed on %s."
msgstr "A telepítő %s rendszerbetöltőt talált %s rendszerben."

#: iw/upgrade_bootloader_gui.py:142
msgid "This is the recommended option."
msgstr "Ez a javasolt beállítás."

#: iw/upgrade_bootloader_gui.py:147
msgid "_Create new boot loader configuration"
msgstr "Ú_j rendszerindító beállítás készítése"

#: iw/upgrade_bootloader_gui.py:149
msgid "This option creates a new boot loader configuration.  If you wish to switch boot loaders, you should choose this."
msgstr "Ezzel új rendszerindító beállítást lehet készíteni.  Ha le szeretné cserélni az eddigi rendszerindítót, válassza ezt."

#: iw/upgrade_bootloader_gui.py:156
msgid "_Skip boot loader updating"
msgstr "Rendszerbetöltő frissítés ki_hagyása"

#: iw/upgrade_bootloader_gui.py:157
msgid "This option makes no changes to boot loader configuration.  If you are using a third party boot loader, you should choose this."
msgstr "Ebben az esetben nem változik meg a rendszerindító beállítás.  Ha külső forrásból származó rendszerindítót használ, akkor válassza ezt a lehetőséget."

#: iw/upgrade_bootloader_gui.py:168
msgid "What would you like to do?"
msgstr "Mit szeretne tenni?"

#: iw/upgrade_migratefs_gui.py:38
#: textw/upgrade_text.py:36
msgid "Migrate File Systems"
msgstr "Fájlrendszerek migrálása"

#: iw/upgrade_migratefs_gui.py:69
#, python-format
msgid ""
"This release of %s supports an updated file system, which has several benefits over the file system traditionally shipped in %s.  This installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"%s jelen kiadása támogat egy frissebb fájlrendszert, ami több szempontból is előnyösebb hagyományosan %s rendszerekben adottaknál.  Ez a telepítő képes a kialakított részeket adatvesztés nélkül átalakítani.\n"
"\n"
"Ezek közül mely részeket szeretné átalakítani?"

#: iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr "Lapozó rész frissítése"

#: iw/upgrade_swap_gui.py:93
#: textw/upgrade_text.py:101
#, python-format
msgid "Recent kernels (2.4 or newer) need significantly more swap than older kernels, up to twice the amount of RAM on the system.  You currently have %dMB of swap configured, but you may create additional swap space on one of your file systems now."
msgstr "A 2.4-es és újabb Linux rendszermagok nagyobb lapozó területet igényelnek, mint a régebbi változatok - ez elérheti a RAM méretének kétszeresét.  Jelenleg %d MB méretű lapozó van beállítva, de most hozzáadhat további lapozó helyet is valamelyik fájl rendszeren."

#: iw/upgrade_swap_gui.py:100
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""
"\n"
"\n"
"A telepítő %s MB RAM-ot talált a gépben.\n"

#: iw/upgrade_swap_gui.py:112
msgid "I _want to create a swap file"
msgstr "Lapo_zó fájlt szeretnék csinálni"

#: iw/upgrade_swap_gui.py:121
msgid "Select the _partition to put the swap file on:"
msgstr "Erre a _partícióra kerüljön a lapozó fájl:"

#: iw/upgrade_swap_gui.py:139
#: textw/upgrade_text.py:119
msgid "Partition"
msgstr "Partíció"

#: iw/upgrade_swap_gui.py:139
msgid "Free Space (MB)"
msgstr "Szabad hely (MB)"

#: iw/upgrade_swap_gui.py:157
#, python-format
msgid "A minimum swap file size of %d MB is recommended.  Please enter a size for the swap file:"
msgstr "A lapozó fájlt érdemes legalább %d MB-ra venni.  Kérem adja meg a lapozó fájl méretét:"

#: iw/upgrade_swap_gui.py:172
msgid "Swap file _size (MB):"
msgstr "Lap_ozó fájl mérete (MB):"

#: iw/upgrade_swap_gui.py:182
msgid "I _don't want to create a swap file"
msgstr "Nem a_karok lapozó fájlt készíteni"

#: iw/upgrade_swap_gui.py:192
msgid "A swap file is strongly recommended. Failure to create one could cause the installer to abort abnormally.  Are you sure you wish to continue?"
msgstr "Mindenképpen azt ajánljuk, hogy készítsen egy lapozó fájlt. Ha ezt nem teszi meg, előfordulhat, hogy a telepítő váratlanul leáll.  Biztosan folytatni kívánja?"

#: iw/upgrade_swap_gui.py:200
#: textw/upgrade_text.py:188
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr "A lapozó fájl méretének 1 és 2000 MB közé kell esnie."

#: iw/upgrade_swap_gui.py:207
#: textw/upgrade_text.py:183
msgid "There is not enough space on the device you selected for the swap partition."
msgstr "Nincs elég hely a kiválasztott eszközön a lapozó rész elkészítéséhez."

#: iw/welcome_gui.py:56
#: textw/welcome_text.py:36
msgid "Network Install Required"
msgstr "Hálózati telepítés szükséges"

#: iw/welcome_gui.py:57
#: textw/welcome_text.py:37
msgid "Your installation source is set to a network location, but no netork devices were found on your system.  To avoid a network installation, boot with the full DVD, full CD set, or do not pass a repo= parameter that specifies a network source."
msgstr "A telepítés forrását hálózati helyre állította, de nem találtam hálózati eszközt a rendszerben.  A hálózati telepítést elkerülendő indítson újra egy teljes DVD vagy CD készlettel, vagy ne adjon meg hálózati forrást jelölő 'repo=' paramétert."

#: iw/welcome_gui.py:67
msgid "E_xit Installer"
msgstr "_Kilépés a telepítőből"

#: iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr "z/IPL rendszerindító beállításai"

#: iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr "z/IPL rendszerindítót telepít a gépre."

#: iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or your setup may require."
msgstr ""
"A z/IPL rendszerindító telepítése következik.\n"
"\n"
"A gyökér rész a telepítés korábbi fázisában kijelölt rész lesz.\n"
"\n"
"A rendszer indítása a kiválasztott alapértelmezett rendszermaggal történik.\n"
"\n"
"Ha később, a telepítés után szeretne módosításokat végrehajtani, akkor módosítsa az /etc/zipl.conf beállítás fájlt.\n"
"\n"
"Adja meg azokat a rendszermag-paramétereket, melyek a gép vagy a konfiguráció használatához szükségesek."

#: iw/zipl_gui.py:90
#: textw/zipl_text.py:71
msgid "Kernel Parameters"
msgstr "Rendszermag paraméterek"

#: iw/zipl_gui.py:93
#: iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr "Chandev paraméterek"

#: loader/cdinstall.c:185
#: loader/cdinstall.c:206
#: loader/mediacheck.c:60
msgid "Media Check"
msgstr "Adathordozó ellenőrzése"

#: loader/cdinstall.c:185
#: loader/cdinstall.c:188
#: loader/cdinstall.c:206
#: loader/cdinstall.c:214
#: loader/method.c:326
msgid "Test"
msgstr "Próba"

#: loader/cdinstall.c:185
#: loader/cdinstall.c:189
msgid "Eject Disc"
msgstr "Lemez kiadása"

#: loader/cdinstall.c:186
#, c-format
msgid "Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject the disc and insert another for testing."
msgstr "Válassza „%s” opciót az olvasóban levő lemez ellenőrzéséhez, vagy „%s” opciót a lemez kivételéhez, és újabb tesztelendő lemez behelyezéséhez."

#: loader/cdinstall.c:207
#, c-format
msgid "If you would like to test additional media, insert the next disc and press \"%s\". Testing each disc is not strictly required, however it is highly recommended.  Minimally, the discs should be tested prior to using them for the first time. After they have been successfully tested, it is not required to retest each disc prior to using it again."
msgstr "Amennyiben további adathordozót is akar még ellenőrizni, helyezze be a következő lemezt, és nyomjon \"%s\" gombot. Minden egyes lemez ellenőrzése nem feltétlenül szükséges, azonban ajánlatos. Legalább az első használat előtt érdemes a lemezeket ellenőrizni. Amennyiben már sikeresen ellenőrizték őket, nem szükséges az újra ellenőrizni."

#: loader/cdinstall.c:229
#, c-format
msgid "The %s disc was not found in any of your drives. Please insert the %s disc and press %s to retry."
msgstr "%s lemez nincs egyik olvasóban sem. Kérem tegye be %s lemezt, és nyomjon %s gombot, hogy újra próbálja."

#: loader/cdinstall.c:248
msgid "Disc Found"
msgstr "Lemez felismerve"

#: loader/cdinstall.c:249
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""
"Nyomjon %s gombot, hogy leellenőrizzem az adathordozót.\n"
"\n"
"Válassza %s gombot, hogy kihagyjam az adathordozó ellenőrzését, és megkezdjem a telepítést."

#: loader/cdinstall.c:339
msgid "Scanning"
msgstr "Beolvasás"

#: loader/cdinstall.c:339
#, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr "Telepítő képeket keresek %s CD eszközön\n"

#: loader/cdinstall.c:341
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr "Telepítő képeket keresek %s CD eszközön"

#: loader/cdinstall.c:442
#, c-format
msgid "The %s disc was not found in any of your CDROM drives. Please insert the %s disc and press %s to retry."
msgstr "Egyik CD-ROM olvasóban sincs %s lemez. Kérem tegyen %s lemezt, és nyomjon %s gombot, hogy újra próbálja."

#: loader/cdinstall.c:451
msgid "Disc Not Found"
msgstr "Lemez nem található"

#: loader/cdinstall.c:452
#: loader/driverdisk.c:244
#: loader/driverdisk.c:275
#: loader/driverdisk.c:359
#: loader/driverselect.c:81
#: loader/driverselect.c:179
#: loader/driverselect.c:205
#: loader/hdinstall.c:222
#: loader/hdinstall.c:279
#: loader/kbd.c:119
#: loader/loader.c:483
#: loader/loader.c:503
#: loader/loader.c:1374
#: loader/loader.c:1407
#: loader/net.c:574
#: loader/net.c:945
#: loader/net.c:1858
#: loader/net.c:1880
#: loader/nfsinstall.c:91
#: loader/urls.c:292
#: storage/__init__.py:136
#: storage/devicetree.py:83
#: textw/constants_text.py:52
msgid "Back"
msgstr "Vissza"

#: loader/cdinstall.c:518
msgid "Cannot find kickstart file on CDROM."
msgstr "Nem találtam kickstart fájlt a CD-n."

#: loader/copy.c:51
#: loader/method.c:280
#, c-format
msgid "Failed to read directory %s: %m"
msgstr "%s mappa beolvasása sikertelen: %m"

#: loader/driverdisk.c:139
msgid "Loading"
msgstr "Betöltés"

#: loader/driverdisk.c:139
msgid "Reading driver disk"
msgstr "Eszközkezelő szoftver lemezének olvasása"

#: loader/driverdisk.c:238
#: loader/driverdisk.c:270
msgid "Driver Disk Source"
msgstr "Eszközkezelő szoftver lemezének forrása"

#: loader/driverdisk.c:239
msgid "You have multiple devices which could serve as sources for a driver disk.  Which would you like to use?"
msgstr "Több olyan meghajtó található a gépben, amely forrásául szolgálhat az eszközkezelő szoftver lemezének.  Melyiket szeretné használni?"

#: loader/driverdisk.c:271
msgid "There are multiple partitions on this device which could contain the driver disk image.  Which would you like to use?"
msgstr "Több olyan partíció található ezen az eszközön, amely tartalmazhatja a eszközkezelő szoftver lemezének képmását.  Melyiket szeretné használni?"

#: loader/driverdisk.c:306
msgid "Failed to mount partition."
msgstr "Nem sikerült becsatolni a partíciót."

#: loader/driverdisk.c:311
msgid "Select driver disk image"
msgstr "Válasszon eszközkezelő szoftver lemez képmását"

#: loader/driverdisk.c:312
msgid "Select the file which is your driver disk image."
msgstr "Válassza ki a fájlt, amelyik a eszközkezelő szoftver lemezének képmása."

#: loader/driverdisk.c:341
msgid "Failed to load driver disk from file."
msgstr "Nem sikerült betölteni a vezérlő képmást a fájlból."

#: loader/driverdisk.c:353
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr "Tegye be a vezérlőprogram lemezt /dev/%s olvasóba, majd nyomjon „OK”-t, hogy folytassam."

#: loader/driverdisk.c:359
msgid "Insert Driver Disk"
msgstr "Tegyen be eszközkezelő szoftver lemezét"

#: loader/driverdisk.c:372
msgid "Failed to mount driver disk."
msgstr "Nem sikerült becsatolni a eszközkezelő szoftver lemezét."

#: loader/driverdisk.c:380
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr "%s jelen kiadásában érvénytelen eszközkezelő szoftver lemeze."

#: loader/driverdisk.c:443
msgid "Manually choose"
msgstr "Kézi kiválasztás"

#: loader/driverdisk.c:444
msgid "Load another disk"
msgstr "Másik lemez betöltése"

#: loader/driverdisk.c:445
msgid "No devices of the appropriate type were found on this driver disk.  Would you like to manually select the driver, continue anyway, or load another driver disk?"
msgstr "Nincs megfelelő típusú eszközkezelő szoftver ezen a lemezen.  Kézzel szeretné kiválasztani a vezérlőprogramot, folytatja mindenképp, vagy betölt egy másik lemezt?"

#: loader/driverdisk.c:483
msgid "Driver disk"
msgstr "Vezérlőprogram lemez"

#: loader/driverdisk.c:484
msgid "Do you have a driver disk?"
msgstr "Van vezérlőprogram lemeze?"

#: loader/driverdisk.c:493
msgid "More Driver Disks?"
msgstr "Van még vezérlőprogram lemez?"

#: loader/driverdisk.c:494
msgid "Do you wish to load any more driver disks?"
msgstr "Szeretne további vezérlő lemezeket betölteni?"

#: loader/driverdisk.c:539
#: loader/driverdisk.c:576
#: loader/hdinstall.c:372
#: loader/kickstart.c:132
#: loader/kickstart.c:142
#: loader/kickstart.c:185
#: loader/kickstart.c:562
#: loader/modules.c:335
#: loader/modules.c:347
#: loader/net.c:1579
#: loader/net.c:1600
#: loader/nfsinstall.c:337
#: loader/urlinstall.c:451
#: loader/urlinstall.c:460
#: loader/urlinstall.c:469
msgid "Kickstart Error"
msgstr "Kickstart hiba"

#: loader/driverdisk.c:540
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr "Ismeretlen vezérlő lemez kickstart forrás: %s"

#: loader/driverdisk.c:577
#, c-format
msgid "The following invalid argument was specified for the kickstart driver disk command: %s:%s"
msgstr "A következő érvénytelen argumentum volt megadva a kickstart vezérlő lemez parancsában: %s:%s"

#: loader/driverselect.c:67
#, c-format
msgid "Please enter any parameters which you wish to pass to the %s module separated by spaces.  If you don't know what parameters to supply, skip this screen by pressing the \"OK\" button."
msgstr "Kérem adja meg %s modulnak átadni kívánt paramétereket, szóközzel elválasztva.  Ha nem tudja, milyen paraméterekre van szükség, lépjen tovább a következő képernyőre „OK” gombot nyomva."

#: loader/driverselect.c:91
msgid "Enter Module Parameters"
msgstr "Modul paraméterek megadása"

#: loader/driverselect.c:178
msgid "No drivers found"
msgstr "Nincs vezérlőprogram"

#: loader/driverselect.c:178
msgid "Load driver disk"
msgstr "Vezérlő lemez betöltése"

#: loader/driverselect.c:179
msgid "No drivers were found to manually insert.  Would you like to use a driver disk?"
msgstr "Nincs kézzel beszúrható vezérlő.  Szeretne vezérlőprogram lemezt használni?"

#: loader/driverselect.c:197
msgid "Please select the driver below which you wish to load.  If it does not appear and you have a driver disk, press F2."
msgstr "Kérem válassza ki alább a betöltendő vezérlőprogramot.  Ha nem jelenik meg, és rendelkezik vezérlő lemez, nyomjon F2-t."

#: loader/driverselect.c:206
msgid "Specify optional module arguments"
msgstr "Választható modul argumentumok megadása"

#: loader/driverselect.c:234
msgid "Select Device Driver to Load"
msgstr "Válassza ki a betöltendő vezérlőprogramot"

#: loader/hdinstall.c:135
msgid "An error occured finding the installation image on your hard drive.  Please check your images and try again."
msgstr "Hiba történt a telepítő képet keresése közben a merevlemezen.  Kérem ellenőrizze a képeket, és próbálkozzon újra."

#: loader/hdinstall.c:223
msgid "You don't seem to have any hard drives on your system! Would you like to configure additional devices?"
msgstr "Úgy tűnik, nincs egyetlen merevlemez vezérlő sem a rendszerben. Kíván további eszközöket beállítani?"

#: loader/hdinstall.c:236
#, c-format
msgid "What partition and directory on that partition holds the installation image for %s?  If you don't see the disk drive you're using listed here, press F2 to configure additional devices."
msgstr "Melyik részen, és azon belül melyik mappában van a telepítő képmás a %s rendszerhez?  Ha nem látja a listában a használt lemezt, nyomjon F2-t további eszközök beállításához."

#: loader/hdinstall.c:263
msgid "Directory holding image:"
msgstr "Képeket tartalmazó mappa:"

#: loader/hdinstall.c:291
msgid "Select Partition"
msgstr "Válasszon partíciót"

#: loader/hdinstall.c:341
#, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr "Úgy tűnik %s eszköz nem tartalmaz telepítő képet."

#: loader/hdinstall.c:373
#, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr "Hibás argumentum %s HD kickstart parancsnál: %s"

#: loader/hdinstall.c:443
#: loader/hdinstall.c:499
msgid "Cannot find kickstart file on hard drive."
msgstr "Nincs kickstart fájl a merevlemezen."

#: loader/hdinstall.c:486
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr "Nincs merevlemez %s BIOS lemezhez"

#: loader/kbd.c:117
msgid "Keyboard Type"
msgstr "Billentyűzet típus"

#: loader/kbd.c:118
msgid "What type of keyboard do you have?"
msgstr "Milyen típusú billentyűzete van?"

#: loader/kickstart.c:133
#, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "%s kickstart fájlt megnyitása közben hiba történt: %m"

#: loader/kickstart.c:143
#, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "%s kickstart fájlt beolvasása közben hiba történt: %m"

#: loader/kickstart.c:186
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "%s hibás: %d. sor %s kickstart fájl."

#: loader/kickstart.c:285
msgid "Cannot find ks.cfg on removable media."
msgstr "Nincs ks.cfg fájl a cserélhető adathordozón."

#: loader/kickstart.c:386
msgid "Unable to download the kickstart file.  Please modify the kickstart parameter below or press Cancel to proceed as an interactive installation."
msgstr "Nem sikerült letölteni a Kickstart-fájlt.  Kérem változtassa meg az alábbi kickstart paramétert, vagy nyomjon Mégsem-et, hogy interaktívan folytassuk a telepítést."

#: loader/kickstart.c:395
msgid "Error downloading kickstart file"
msgstr "Kickstart fájlt letöltése hibás"

#: loader/kickstart.c:563
#, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr "Hibás paraméter a kickstart metódus parancs leállításában: %s: %s"

#: loader/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "Üdvözöli %s %s – Mentő mód"

#: loader/lang.c:65
#: loader/loader.c:226
msgid "  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr "  <Tab>/<AltTab> elemek között  |  <Szóköz> megjelölés  |  <F12> köv. képernyő  "

#: loader/lang.c:376
msgid "Choose a Language"
msgstr "Válasszon nyelvet"

#: loader/loader.c:127
msgid "Local CD/DVD"
msgstr "Helyi CD/DVD"

#: loader/loader.c:128
msgid "Hard drive"
msgstr "Merevlemez"

#: loader/loader.c:129
msgid "NFS directory"
msgstr "NFS mappa"

#: loader/loader.c:434
#: loader/loader.c:478
msgid "Update Disk Source"
msgstr "Frissítő lemez forrása"

#: loader/loader.c:435
msgid "You have multiple devices which could serve as sources for an update disk.  Which would you like to use?"
msgstr "Több eszköz van a rendszerben, ami tartalmazhat frissítő lemezt. Melyiket szeretné használni?"

#: loader/loader.c:479
msgid "There are multiple partitions on this device which could contain the update disk image.  Which would you like to use?"
msgstr "Több rész van ezen az eszközön, ami tartalmazhat frissítő lemez képmást.  Melyiket szeretné használni?"

#: loader/loader.c:497
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr "Tegye be a frissítés lemezt %s eszközbe, majd nyomjon „OK”-t, hogy folytassam."

#: loader/loader.c:503
msgid "Updates Disk"
msgstr "Frissítő lemez"

#: loader/loader.c:521
msgid "Failed to mount updates disk"
msgstr "Nem sikerült becsatolni a frissítő lemezt."

#: loader/loader.c:526
msgid "Updates"
msgstr "Frissítések"

#: loader/loader.c:526
msgid "Reading anaconda updates"
msgstr "Anaconda frissítések olvasása"

#: loader/loader.c:562
msgid "Unable to download the updates image.  Please modify the updates location below or press Cancel to proceed without updates.."
msgstr "Nem sikerült letölteni a frissítő képmást.  Kérem módosítsa alább a frissítések helyét, vagy nyomjon Mégsem-et, hogy frissítések nélkül folytatódjon a telepítés..."

#: loader/loader.c:571
msgid "Error downloading updates image"
msgstr "Hiba a frissítő képmás letöltése közben"

#: loader/loader.c:1183
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "Nincs elég memória ebben a gépben %s telepítéséhez."

#: loader/loader.c:1240
msgid "Media Detected"
msgstr "Adathordozó érzékelve"

#: loader/loader.c:1241
msgid "Found local installation media"
msgstr "Helyi telepítő adathordozó érzékelve"

#: loader/loader.c:1366
msgid "Rescue Method"
msgstr "Mentő eljárás"

#: loader/loader.c:1367
msgid "Installation Method"
msgstr "Telepítő eljárás"

#: loader/loader.c:1369
msgid "What type of media contains the rescue image?"
msgstr "Milyen típusú hordozón van a mentő képmás?"

#: loader/loader.c:1371
msgid "What type of media contains the installation image?"
msgstr "Milyen típusú hordozón van a telepítő képmás?"

#: loader/loader.c:1406
msgid "No driver found"
msgstr "Nincs vezérlő"

#: loader/loader.c:1406
msgid "Select driver"
msgstr "Válasszon vezérlőt"

#: loader/loader.c:1407
msgid "Use a driver disk"
msgstr "Vezérlő lemez használata"

#: loader/loader.c:1408
msgid "Unable to find any devices of the type needed for this installation type.  Would you like to manually select your driver or use a driver disk?"
msgstr "Nem találtam a telepítés típusának megfelelő eszközt.  Szeretne vezérlőt választani, vagy vezérlő lemezt használni?"

#: loader/loader.c:1638
msgid "The following devices have been found on your system."
msgstr "A következő eszközöket találtam a rendszerben."

#: loader/loader.c:1640
msgid "No device drivers have been loaded for your system.  Would you like to load any now?"
msgstr "Nincs eszköz vezérlő betöltve a rendszerbe.  Be szeretne most tölteni egyet?"

#: loader/loader.c:1644
msgid "Devices"
msgstr "Eszközök"

#: loader/loader.c:1645
msgid "Done"
msgstr "Kész"

#: loader/loader.c:1646
msgid "Add Device"
msgstr "Eszköz hozzáadása"

#: loader/loader.c:1858
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr "betöltő már lefutott.  Parancssor indítása.\n"

#: loader/loader.c:2236
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr "Anaconda %s, a %s mentő mód fut – kérem várjon.\n"

#: loader/loader.c:2238
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr "Anaconda %s, a %s rendszer telepítő fut – kérem várjon.\n"

#: loader/mediacheck.c:46
#, c-format
msgid "Unable to find install image %s"
msgstr "Nem találtam e telepítési képmást: %s"

#: loader/mediacheck.c:52
#, c-format
msgid "Checking \"%s\"."
msgstr "„%s” ellenőrzése."

#: loader/mediacheck.c:54
#, c-format
msgid "Checking media."
msgstr "Adathordozó ellenőrzése."

#: loader/mediacheck.c:86
msgid "Unable to read the disc checksum from the primary volume descriptor.  This probably means the disc was created without adding the checksum."
msgstr "Az elsődleges kötet leíróból nem sikerült kiolvasni a lemez ellenőrző összegét.  Ez valószínűleg azt jelenti, hogy a lemezt ellenőrző összeg nélkül készítették."

#: loader/mediacheck.c:93
msgid "The image which was just tested has errors. This could be due to a corrupt download or a bad disc.  If applicable, please clean the disc and try again.  If this test continues to fail you should not continue the install."
msgstr "A tesztelt képmás hibás. Ezt okozhatta egy letöltési hiba vagy lemezhiba.  Ha érdemes, tisztítsa meg a lemezt, és próbálja újra.  Ne folytassa a telepítést, ha a teszt továbbra sem sikeres."

#: loader/mediacheck.c:101
msgid "Success"
msgstr "Sikerült"

#: loader/mediacheck.c:102
msgid "The image which was just tested was successfully verified.  It should be OK to install from this media.  Note that not all media/drive errors can be detected by the media check."
msgstr "A lemezképet – amit éppen kipróbált – sikeresen ellenőriztem.  Erről a adathordozóról nyugodtan telepíthet.  Figyelem: nem tud minden hordozó/olvasó hibát érzékelni a lemezellenőrzés."

#: loader/method.c:323
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"El szeretné végezni az ellenőrző összeg vizsgálatát ezen ISO képmásra:\n"
"\n"
"   %s?"

#: loader/method.c:326
msgid "Checksum Test"
msgstr "Ellenőrző összeg vizsgálata"

#: loader/modules.c:336
#, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr "%s device kickstart parancsban hibás argumentum: %s"

#: loader/modules.c:348
msgid "A module name must be specified for the kickstart device command."
msgstr "Modul nevet is meg kell adni a kickstart device parancsában."

#: loader/net.c:111
msgid "Invalid Prefix"
msgstr "Érvénytelen előtag"

#: loader/net.c:112
msgid "Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for IPv6 networks"
msgstr "Az előtag 1 és 32 közé kell essen IPv4 hálózatnál, vagy 1 és 128 közé IPv6 hálózatnál."

#: loader/net.c:464
#: loader/net.c:520
msgid "Network Error"
msgstr "Hálózat hiba"

#: loader/net.c:465
#: loader/net.c:521
msgid "There was an error configuring your network interface."
msgstr "Hálózat csatoló beállítása alatt hiba történt."

#: loader/net.c:552
#: textw/netconfig_text.py:131
msgid "Enable IPv4 support"
msgstr "IPv4 támogatás engedélyezése"

#: loader/net.c:566
msgid "Enable IPv6 support"
msgstr "IPv6 támogatás engedélyezése"

#: loader/net.c:606
msgid "Configure TCP/IP"
msgstr "TCP/IP beállítás"

#: loader/net.c:663
msgid "Missing Protocol"
msgstr "Hiányzó protokoll"

#: loader/net.c:664
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr "Legalább egy protokollt (IPv4 vagy IPv6) ki kell választani."

#: loader/net.c:671
msgid "IPv4 Needed for NFS"
msgstr "NFS-hez IPv4 kell"

#: loader/net.c:672
msgid "NFS installation method requires IPv4 support."
msgstr "NFS telepítő eljárás IPv4 támogatást igényel."

#: loader/net.c:771
msgid "IPv4 address:"
msgstr "IPv4 cím:"

#: loader/net.c:783
#: loader/net.c:850
#: ui/netconfig.glade.h:1
msgid "/"
msgstr "/"

#: loader/net.c:838
msgid "IPv6 address:"
msgstr "IPv6 cím:"

#: loader/net.c:904
#: textw/netconfig_text.py:158
msgid "Gateway:"
msgstr "Átjáró:"

#: loader/net.c:912
msgid "Name Server:"
msgstr "Névkiszolgáló:"

#: loader/net.c:951
msgid "Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr "Adjon meg IPv4 és/vagy IPv6 címet és előtagot (cím / előtag).  IPv4 esetén használható a négytagú, pontokkal elválasztott hálózat maszk, vagy a CIDR stílusú előtag. Az átjáró és a névkiszolgáló mezőkben érvényes IPv4 vagy IPv6 címeknek kell szerepelni."

#: loader/net.c:971
msgid "Manual TCP/IP Configuration"
msgstr "Kézi TCP/IP beállítás"

#: loader/net.c:1094
#: loader/net.c:1101
msgid "Missing Information"
msgstr "Hiányzó információ"

#: loader/net.c:1095
msgid "You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr "Meg kell adnia egy érvényes IPv4 címet, valamint vagy egy hálózat maszkot, vagy egy CIDR-előtagot."

#: loader/net.c:1102
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr "Meg kell adni egy érvényes IPv6 címet és egy CIDR-előtagot."

#: loader/net.c:1580
#, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr "%s network kickstart parancsban hibás argumentum: %s"

#: loader/net.c:1601
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr "Network parancsban hibás a bootproto %s"

#: loader/net.c:1678
msgid "Seconds:"
msgstr "Másodpercek:"

#: loader/net.c:1853
msgid "Networking Device"
msgstr "Hálózat eszköz"

#: loader/net.c:1854
msgid "You have multiple network devices on this system. Which would you like to install through?"
msgstr "Több hálózat eszköz van a rendszerben. Melyiken át szeretné elvégezni a telepítést?"

#: loader/net.c:1858
msgid "Identify"
msgstr "Azonosítás"

#: loader/net.c:1867
msgid "You can identify the physical port for"
msgstr "Azonosíthat egy fizikai kaput "

#: loader/net.c:1869
msgid "by flashing the LED lights for a number of seconds.  Enter a number between 1 and 30 to set the duration to flash the LED port lights."
msgstr "eszközhöz a LED lámpácskát villogtatva egy megadott ideig.  Adjon meg egy számot 1 és 30 között, hogy beállítsa a LED kapu fény villogás idejét."

#: loader/net.c:1879
msgid "Identify NIC"
msgstr "Hálózat csatoló azonosítása"

#: loader/net.c:1892
msgid "Invalid Duration"
msgstr "Érvénytelen időtartam"

#: loader/net.c:1893
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr "Meg kell adnia a másodpercek számát 1 és 30 közötti egészként."

#: loader/net.c:1905
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr "%s kapu fények villogtatása %d másodpercig."

#: loader/net.c:2074
#: loader/net.c:2078
#, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr "Várok, hogy NetworkManager beállítsa %s eszközt.\n"

#: loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr "NFS kiszolgáló név:"

#: loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr "%s mappa:"

#: loader/nfsinstall.c:83
#, c-format
msgid "Please enter the server name and path to your %s installation image."
msgstr "Kérem adja meg a kiszolgáló nevét és az utat az ön %s telepítő képéhez."

#: loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr "NFS beállítás"

#: loader/nfsinstall.c:251
msgid "That directory could not be mounted from the server."
msgstr "Azt a mappát nem sikerült becsatolni"

#: loader/nfsinstall.c:262
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr "Úgy tűnik, az a mappa nem tartalmaz %s telepítő képet."

#: loader/nfsinstall.c:338
#, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr "%s NFS kickstart parancsban hibás argumentum: %s"

#: loader/telnetd.c:89
#: loader/telnetd.c:128
msgid "Telnet"
msgstr "Telnet"

#: loader/telnetd.c:89
msgid "Waiting for telnet connection."
msgstr "Várakozás telnet kapcsolatra."

#: loader/telnetd.c:128
msgid "Running anaconda via telnet."
msgstr "Anaconda futtatása telneten át."

#: loader/urlinstall.c:84
#, c-format
msgid "Unable to retrieve %s://%s%s."
msgstr "Nem tudok letölteni %s://%s/%s helyről."

#: loader/urlinstall.c:176
msgid "Unable to retrieve the install image."
msgstr "Nem tudtam letölteni a telepítő képet."

#: loader/urlinstall.c:452
#, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr "%s Url kickstart parancsban hibás argumentum: %s"

#: loader/urlinstall.c:461
msgid "Must supply a --url argument to Url kickstart method."
msgstr "Meg kell adni egy --url argumentumot az Url kickstart parancsnak."

#: loader/urlinstall.c:470
#, c-format
msgid "Unknown Url method %s"
msgstr "Ismeretlen Url parancs %s"

#: loader/urls.c:235
#: loader/urls.c:237
#: loader/urls.c:240
msgid "Retrieving"
msgstr "Letöltés"

#: loader/urls.c:295
#, c-format
msgid "Please enter the URL containing the %s installation image on your server."
msgstr "Kérem, adja meg az URL-t, ami %s telepítő képeket tartalmazza a kiszolgálón."

#: loader/urls.c:321
msgid "URL Setup"
msgstr "URL beállítás"

#: loader/urls.c:329
msgid "You must enter a URL."
msgstr "Meg kell adni egy URL-t."

#: loader/urls.c:335
msgid "URL must be either an ftp or http URL"
msgstr "Az URL vagy ftp vagy http URL kell legyen."

#: loader/urls.c:346
msgid "Unknown Host"
msgstr "Ismeretlen gazda"

#: loader/urls.c:347
#, c-format
msgid "%s is not a valid hostname."
msgstr "%s nem érvényes gazdanév."

#: loader/windows.c:65
msgid "Loading SCSI driver"
msgstr "SCSI vezérlő betöltése"

#: loader/windows.c:66
#, c-format
msgid "Loading %s driver"
msgstr "%s vezérlő betöltése"

#: storage/__init__.py:87
msgid "Unknown Device"
msgstr "Ismeretlen eszköz"

#: storage/__init__.py:88
#, python-format
msgid "The installation source given by device %s could not be found.  Please check your parameters and try again."
msgstr "A %s eszköz telepítő forrást nem találom.  Kérem ellenőrizze a paramétereket, és próbálja újra."

#: storage/__init__.py:99
msgid "Installation cannot continue."
msgstr "A telepítés nem folytatható."

#: storage/__init__.py:100
msgid "The storage configuration you have chosen has already been activated. You can no longer return to the disk editing screen. Would you like to continue with the installation process?"
msgstr "A választott felosztás opciók már élnek. Már nem léphet vissza a lemez szerkesztő képernyőre. Folytatni szeretné a telepítést?"

#: storage/__init__.py:128
msgid "Encrypt device?"
msgstr "Eszköz titkosítása?"

#: storage/__init__.py:129
msgid "You specified block device encryption should be enabled, but you have not supplied a passphrase. If you do not go back and provide a passphrase, block device encryption will be disabled."
msgstr "Megadta, hogy a blokk eszköz titkosítás engedélyezett legyen, de nem adott meg jelszót. Ha nem ad meg egy jelszót, akkor a blokk eszköz titkosítása letiltásra kerül."

#: storage/__init__.py:152
msgid "Writing storage configuration to disk"
msgstr "Felosztás lemezre írása"

#: storage/__init__.py:153
msgid "The partitioning options you have selected will now be written to disk.  Any data on deleted or reformatted partitions will be lost."
msgstr "Az ön által kiválasztott partícionálási beállításokat most lemezre írom.  A törölt illetve újraformázott partíciókon meglévő adatok elvesznek."

#: storage/__init__.py:158
msgid "Go _back"
msgstr "_Vissza"

#: storage/__init__.py:159
msgid "_Write changes to disk"
msgstr "Vált_ozások lemezre írása"

#: storage/__init__.py:172
msgid "Running..."
msgstr "Futtatás alatt..."

#: storage/__init__.py:173
msgid "Storing encryption keys"
msgstr "Titkosítási kulcsok mentése"

#: storage/__init__.py:191
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr "Hiba egy titkosítási kulcs letárolása közben: %s\n"

#: storage/__init__.py:307
msgid "Finding Devices"
msgstr "Eszközök keresése"

#: storage/__init__.py:308
msgid "Finding storage devices"
msgstr "Tároló eszközök keresése"

#: storage/__init__.py:549
msgid "This partition is holding the data for the hard drive install."
msgstr "E rész tartalmazza a merevlemezes telepítéshez szükséges adatokat."

#: storage/__init__.py:554
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "LDL DASD részt nem lehet törölni."

#: storage/__init__.py:560
#, python-format
msgid "This device is part of the RAID device %s."
msgstr "E rész a %s RAID eszközhöz tartozik."

#: storage/__init__.py:563
msgid "This device is part of a RAID device."
msgstr "E rész egy RAID eszközhöz tartozik."

#: storage/__init__.py:568
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "Ez az eszköz „%s” LVM kötet csoporthoz tartozik."

#: storage/__init__.py:571
msgid "This device is part of a LVM volume group."
msgstr "E rész egy LVM kötetcsoporthoz tartozik."

#: storage/__init__.py:587
msgid ""
"This device is an extended partition which contains logical partitions that cannot be deleted:\n"
"\n"
msgstr ""
"Ez az eszköz olyan kiterjesztett rész, ami nem törölhető logikai részeket tartalmaz:\n"
"\n"

#: storage/__init__.py:856
#, python-format
msgid "You have not defined a root partition (/), which is required for installation of %s to continue."
msgstr "Nem adta meg a gyökér részt (/), ami nélkül %s nem telepíthető."

#: storage/__init__.py:861
#, python-format
msgid "Your root partition is less than 250 megabytes which is usually too small to install %s."
msgstr "A gyökér rész mérete kisebb 250 MB-nál, ami általában nem elég %s telepítéséhez."

#: storage/__init__.py:867
#, python-format
msgid "Your / partition is less than %s MB which is lower than recommended for a normal %s install."
msgstr "A / rész kisebb %s MB-nál, ami kevesebb, mint a normál %s telepítésre ajánlott minimum."

#: storage/__init__.py:877
#, python-format
msgid "Your / partition does not match the the live image you are installing from.  It must be formatted as %s."
msgstr "A / rész nem egyezik meg az élő képpel, ahonnan telepít.  %s fájlrendszert kell kialakítani."

#: storage/__init__.py:884
#, python-format
msgid "Your %s partition is less than %s megabytes which is lower than recommended for a normal %s install."
msgstr "%s rész kisebb %s megabájtnál, ami kevesebb, mint a normál %s telepítésre ajánlott minimum."

#: storage/__init__.py:911
msgid "Installing on a USB device.  This may or may not produce a working system."
msgstr "Telepítés USB eszközre.  Ez a megoldás működésképtelen rendszert is eredményezhet."

#: storage/__init__.py:914
msgid "Installing on a FireWire device.  This may or may not produce a working system."
msgstr "Telepítés FireWire eszközre.  Ez a megoldás működésképtelen rendszert is eredményezhet."

#: storage/__init__.py:921
msgid "You have not specified a swap partition.  Due to the amount of memory present, a swap partition is required to complete installation."
msgstr "Nem adott meg lapozó részt.  A memória mérete miatt, lapozó rész kell a telepítés befejezéséhez."

#: storage/__init__.py:926
msgid "You have not specified a swap partition.  Although not strictly required in all cases, it will significantly improve performance for most installations."
msgstr "Nem adott meg lapozó részt.  Erre nincs mindig szükség, de a legtöbb esetben jelentősen növelheti a teljesítményt."

#: storage/__init__.py:933
#, python-format
msgid "This mount point is invalid.  The %s directory must be on the / file system."
msgstr "Ez a csatoló pont érvénytelen.  %s mappának a / fájlrendszerben kell lennie."

#: storage/__init__.py:937
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "%s csatoló pontnak linuxos fájlrendszerben kell lennie."

#: storage/__init__.py:948
msgid "No Drives Found"
msgstr "Nincs tároló eszköz"

#: storage/__init__.py:949
msgid "An error has occurred - no valid devices were found on which to create new file systems. Please check your hardware for the cause of this problem."
msgstr "Hiba történt – nincs érvényes eszköz, amin új fájlrendszert lehetne kialakítani. Kérem ellenőrizze a hardvert, hogy kiderítse a probléma okát."

#: storage/__init__.py:1181
#: storage/__init__.py:1190
msgid "Dirty File Systems"
msgstr "Rendezetlen fájlrendszerek"

#: storage/__init__.py:1182
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted cleanly.  Please boot your Linux installation, let the file systems be checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""
"A következő fájlrendszereket nem tisztán csatolták ki.  Indítsa el a rendszert, hagyja, hogy a fájlrendszerek ellenőrzése megtörténjen, majd tisztán állítsa le a rendszert a frissítés  megkezdése előtt.\n"
"%s"

#: storage/__init__.py:1191
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"A rendszer következő részeit nem tisztán csatolták ki.  Biztosan be szeretné csatolni őket?\n"
"%s"

#: storage/__init__.py:1663
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for swap space, you must reformat as a new-style Linux swap partition."
msgstr ""
"A lapozó eszköz:\n"
"\n"
"     %s\n"
"\n"
"egy régi típusú Linux lapozó partíció. Ha ezt az eszközt kívánja használni lapozóként, újra kell formáznia egy új típusúra."

#: storage/__init__.py:1674
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, which means your system is hibernating. To perform an upgrade, please shut down your system rather than hibernating it."
msgstr ""
"     %s\n"
"\n"
"lapozó eszközt az /etc/fstab fájl szerint jelenleg szoftveres felfüggesztéshez használják, ami annyit jelent, hogy a rendszer hibernált állapotban van. A frissítés elvégzéséhez inkább állítsa le a rendszert, ne hibernálja."

#: storage/__init__.py:1682
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, which means your system is hibernating. If you are performing a new install, make sure the installer is set to format all swap devices."
msgstr ""
"     %s\n"
"\n"
"lapozó eszközt az /etc/fstab fájl szerint jelenleg szoftveres felfüggesztéshez használják, ami annyit jelent, hogy a rendszer hibernált állapotban van. Ha új telepítést végez, győződjön meg arról, hogy a telepítő újra alakítja az összes lapozó területet."

#: storage/__init__.py:1694
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a support swap volume.  In order to continue installation, you will need to format the device or skip it."
msgstr ""
"A lapozó eszköz:\n"
"\n"
"     %s\n"
"\n"
"nem tartalmaz egy támogató lapozó kötetet. A telepítés folytatásához, szügsége lesz az eszköz formázására vagy kihagyására."

#: storage/__init__.py:1705
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""
"Hiba történt %s lapozó eszköz engedélyezése alatt: %s\n"
"\n"
"A frissítendő partíción levő /etc/fstab fájl nem hivatkozik érvényes lapozó területre.\n"
"\n"
"Nyomjon OK-t, hogy kilépjen a telepítőből."

#: storage/__init__.py:1711
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"%s lapozó eszköz engedélyezése közben hiba történt: %s\n"
"\n"
"A legvalószínűbb, hogy a lapozó terület még nincs kialakítva.\n"
"\n"
"Nyomjon OK-t, hogy kilépjen a telepítőből."

#: storage/__init__.py:1771
#: storage/__init__.py:1781
msgid "Invalid mount point"
msgstr "Érvénytelen becsatoló pont"

#: storage/__init__.py:1772
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s előkészítése közben hiba történt.  Az elérési út néhány eleme nem mappa. Ez végzetes hiba. A telepítés nem folytatható.\n"
"\n"
"Nyomjon <Enter>-t, hogy kilépjen a telepítőből."

#: storage/__init__.py:1782
#, python-format
msgid ""
"An error occurred when trying to create %s: %s.  This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s előkészítése közben hiba történt: %s.  Ez végzetes hiba. A telepítés nem folytatható.\n"
"\n"
"Nyomjon <Enter>-t, hogy kilépjen a telepítőből."

#: storage/__init__.py:1795
#: storage/__init__.py:1816
msgid "Unable to mount filesystem"
msgstr "Nem lehet fájlrendszert becsatolni"

#: storage/__init__.py:1796
#, python-format
msgid "An error occurred mounting device %s as %s.  You may continue installation, but there may be problems."
msgstr "%s eszközt %s ponton való becsatolása közben hiba történt.  Folytathatja a telepítést, de további hibák léphetnek fel."

#: storage/__init__.py:1817
#, python-format
msgid ""
"An error occurred mounting device %s as %s: %s. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s eszközt %s típusú részként becsatolva hiba történt: %s. Ez végzetes hiba. A telepítés nem folytatható.\n"
"\n"
"Nyomjon <Enter>-t, hogy kilépjen a telepítőből."

#: storage/devicelibs/lvm.py:319
#, python-format
msgid "vginfo failed for %s"
msgstr "vginfo sikertelen %s eszközre"

#: storage/devicelibs/lvm.py:348
#, python-format
msgid "lvs failed for %s"
msgstr "lvs sikertelen %s eszközre"

#: storage/devicetree.py:92
msgid "Confirm"
msgstr "Megerősítés"

#: storage/devicetree.py:93
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during installation."
msgstr ""
"Biztosan ki akarja hagyni a jelszó megadását %s eszközhöz?\n"
"\n"
"Ha átugorja e lépést, az eszköz tartalma elérhetetlen lesz a telepítés közben."

#: storage/devicetree.py:144
#, python-format
msgid ""
"Error processing drive:\n"
"\n"
"%s\n"
"%-0.fMB\n"
"%s\n"
"\n"
"This device may need to be reinitialized.\n"
"\n"
"REINITIALIZING WILL CAUSE ALL DATA TO BE LOST!%s"
msgstr ""
"HIba a meghajtó feldolgozásakor:\n"
"\n"
"%s\n"
"%-0.fMB\n"
"%s\n"
"\n"
"Talán újra kellene inicializálni.\n"
"\n"
"ÚRAINICIALIZÁLÁSKOR MINDEN ADAT EL FOG VESZNI AZ ESZKÖZÖN!%s"

#: storage/devicetree.py:150
msgid "_Ignore drive"
msgstr "Eszköz _mellőzése"

#: storage/devicetree.py:151
msgid "_Re-initialize drive"
msgstr "Eszköz új_rainicializálása"

#: storage/devicetree.py:171
#, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %s.  You can reinitialize all related PVs (%s) which will erase the LVM metadata, or ignore which will preserve the contents."
msgstr ""
"LVM-et feldolgozása közben hiba történt.\n"
"Úgy tűnik ellentmondásos LVM adatok vannak a(z) %s eszközön.  Újra lehet alakítani minden kapcsolódó PV-t (%s) (fizikai kötetet), ami minden LVM leíró adatot töröl, vagy mellőzni amivel megőrzi a tartalmat."

#: storage/devicetree.py:176
msgid "_Ignore"
msgstr "_Mellőz"

#: storage/devicetree.py:177
msgid "_Re-initialize"
msgstr "Újra _alakít"

#: storage/devicetree.py:1730
#: storage/devicetree.py:1779
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr "E rész egy ellentmondásos LVM kötetcsoporthoz tartozik."

#: storage/formats/fs.py:96
msgid "filesystem configuration missing a type"
msgstr "fájlrendszer beállításból hiányzik a típus"

#: storage/formats/fs.py:326
msgid "Formatting"
msgstr "Formázás"

#: storage/formats/fs.py:327
#, python-format
msgid "Creating filesystem on %s"
msgstr "Fájlrendszer kialakítása %s partíción"

#: storage/formats/fs.py:423
msgid "Resizing"
msgstr "Átméretezés"

#: storage/formats/fs.py:424
#, python-format
msgid "Resizing filesystem on %s"
msgstr "Fájlrendszer átméretezése %s partíción"

#: storage/formats/fs.py:467
msgid "Checking"
msgstr "Ellenőrzés"

#: storage/formats/fs.py:468
#, python-format
msgid "Checking filesystem on %s"
msgstr "Fájlrendszer ellenőrzése %s partíción"

#: storage/iscsi.py:82
#: storage/iscsi.py:83
msgid "Scanning iSCSI nodes"
msgstr "iSCSI elemek lekérése"

#: storage/iscsi.py:160
#: storage/iscsi.py:161
msgid "Initializing iSCSI initiator"
msgstr "Az iSCSI inicializáló beállítása"

#: storage/iscsi.py:197
msgid "iSCSI not available"
msgstr "iSCSI nem érhető el"

#: storage/iscsi.py:199
msgid "No initiator name set"
msgstr "Nincs kezdeményező név beállítva"

#: storage/iscsi.py:213
msgid "No iSCSI nodes discovered"
msgstr "Nem találtam iSCSI elemeket"

#: storage/iscsi.py:216
#: storage/iscsi.py:217
msgid "Logging in to iSCSI nodes"
msgstr "Bejelentkezés iSCSI elembe"

#: storage/iscsi.py:239
msgid "No new iSCSI nodes discovered"
msgstr "Nem találtam új iSCSI elemeket"

#: storage/iscsi.py:242
msgid "Could not log in to any of the discovered nodes"
msgstr "Nem tudtam bejelentkezni egyik felfedezett elembe sem"

#: storage/partitioning.py:184
msgid "Could not find enough free space for automatic partitioning.  Press 'OK' to exit the installer."
msgstr "Nem találtam elég szabad helyet az automata particionáláshoz. Nyomjon egy 'OK'-t a telepítőből való kilpéshez."

#: storage/partitioning.py:187
msgid "Could not find enough free space for automatic partitioning, please use another partitioning method."
msgstr "Nem találtam elég szabad helyet automata particionáláshoz. Kérem használjon más particionálási eljárást."

#: storage/partitioning.py:216
msgid "Warnings During Automatic Partitioning"
msgstr "Figyelmeztetések az automata particionálás során"

#: storage/partitioning.py:218
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"A következő figyelmeztetések érkeztek az automata particionálás közben:\n"
"\n"
"%s"

#: storage/partitioning.py:230
#: storage/partitioning.py:250
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"Nyomjon „OK”-t, hogy kilépjen a telepítőből."

#: storage/partitioning.py:232
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%s.%s"
msgstr ""
"Nem sikerült lefoglalni a kért partíciókat: \n"
"\n"
"%s.%s"

#: storage/partitioning.py:252
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""
"\n"
"\n"
"Nyomjon „OK” gombot, hogy más partícionálási lehetőséget válasszon."

#: storage/partitioning.py:254
msgid "Automatic Partitioning Errors"
msgstr "Az Automatikus Particionálás során fellépett hibák"

#: storage/partitioning.py:255
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the installation. %s"
msgstr ""
"A következő hibák jelentkeztek a particionálás miatt:\n"
"\n"
"%s\n"
"\n"
"Ez akkor fordulhat elő, ha nincs elég szabad hely a lemezeken a telepítéshez. %s"

#: storage/partitioning.py:266
msgid "Unrecoverable Error"
msgstr "Helyrehozhatatlan hiba"

#: storage/zfcp.py:50
msgid "You have not specified a device number or the number is invalid"
msgstr "Nem adott meg eszközszámot, vagy a szám érvénytelen"

#: storage/zfcp.py:52
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr "Nem adott meg WWPN port nevet, vagy a név érvénytelen."

#: storage/zfcp.py:54
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "Nem adott meg FCP LUN számot, vagy a szám érvénytelen."

#: storage/zfcp.py:132
#, python-format
msgid "Could not free zFCP device %s from device ignore list (%s)."
msgstr "Nem tudom a %s zFCP eszközt felszabadítani a mellőzött eszközlistáról (%s)"

#: storage/zfcp.py:137
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr "%s zFCP eszköz nem található, még a mellőzött eszközlistán sem."

#: storage/zfcp.py:150
#, python-format
msgid "Could not set zFCP device %s online (%s)."
msgstr "A zFCP eszköz nem beállítható %s online (%s)"

#: storage/zfcp.py:161
#, python-format
msgid "Could not add WWPN %s to zFCP device %s (%s)."
msgstr "Hiba: Nem lehet %s WWPN-t hozzáadni a zFCP eszközhöz %s (%s)."

#: storage/zfcp.py:165
#, python-format
msgid "WWPN %s not found at zFCP device %s."
msgstr "%s WWPN nem található az zFCP eszközben %s."

#: storage/zfcp.py:179
#, python-format
msgid "Could not add LUN %s to WWPN %s on zFCP device %s (%s)."
msgstr "Hiba: %s LUN nem hozzáadható a %s WWPN zFCP eszközön %s (%s)."

#: storage/zfcp.py:183
#, python-format
msgid "LUN %s at WWPN %s on zFCP device %s already configured."
msgstr "%s LUN a WWPN %s az %s zFCP eszközön  már beállított."

#: storage/zfcp.py:193
#, python-format
msgid "Could not read failed attribute of LUN %s at WWPN %s on zFCP device %s (%s)."
msgstr "Nem sikerült a hibás %s LUN attribútum kiolvasása %s WWPN-en zFCP eszközön %s (%s)"

#: storage/zfcp.py:198
#, fuzzy, python-format
msgid "Failed LUN %s at WWPN %s on zFCP device %s removed again."
msgstr ""

#: storage/zfcp.py:253
#, python-format
msgid "Could not correctly delete SCSI device of zFCP %s %s %s (%s)."
msgstr "Precízen nem törölhető a zFCP SCSI eszköze %s %s %s (%s)"

#: storage/zfcp.py:260
#, python-format
msgid "Could not remove LUN %s at WWPN %s on zFCP device %s (%s)."
msgstr "Hiba: %s LUN nem eltávolítható a %s WWPN zFCP eszközön %s (%s)."

#: storage/zfcp.py:275
#, python-format
msgid "Could not remove WWPN %s on zFCP device %s (%s)."
msgstr "Hiba: WWPN %s nem eltávolítható az zFCP eszközön %s (%s)."

#: storage/zfcp.py:299
#, python-format
msgid "Could not set zFCP device %s offline (%s)."
msgstr "Hiba: zFCP eszköz %s kikapcsolva (%s), nem beállítható"

#: textw/complete_text.py:32
msgid "Reboot"
msgstr "Újraindítás"

#: textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr "<Enter>, hogy kilépjen"

#: textw/complete_text.py:62
msgid "Complete"
msgstr "Kész"

#: textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "Billentyűzet kiválasztás"

#: textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr "Milyen típusú billentyűzet van a géphez kötve?"

#: textw/netconfig_text.py:41
#, python-format
msgid "A value is required for the field %s"
msgstr "Érték kell %s mezőnek"

#: textw/netconfig_text.py:86
#: ui/netconfig.glade.h:9
msgid "Enable network interface"
msgstr "Hálózati csatoló engedélyezése"

#: textw/netconfig_text.py:89
#: ui/netconfig.glade.h:10
msgid "This requires that you have an active network connection during the installation process.  Please configure a network interface."
msgstr "Ehhez szükséges, hogy a telepítés alatt legyen aktív hálózat kapcsolat.  Kérem állítson be egy hálózat csatolót."

#: textw/netconfig_text.py:128
msgid "Use dynamic IP configuration (DHCP)"
msgstr "Változó IP beállítás használata (DHCP)"

#: textw/netconfig_text.py:138
msgid "IPv4 Address:"
msgstr "IPv4 cím:"

#: textw/netconfig_text.py:161
msgid "Nameserver:"
msgstr "Névkiszolgáló:"

#: textw/netconfig_text.py:191
msgid "Missing Device"
msgstr "Hiányzó eszköz"

#: textw/netconfig_text.py:192
msgid "You must select a network device"
msgstr "Ki kell választani egy hálózat eszközt"

#: textw/netconfig_text.py:251
msgid "IPv4 Network Mask "
msgstr "IPv4 hálózat maszk"

#: textw/netconfig_text.py:274
msgid "Configuring Network Interfaces"
msgstr "Hálózat csatolók beállítása"

#: textw/netconfig_text.py:274
msgid "Waiting for NetworkManager"
msgstr "Várakozás a NetworkManager-re"

#: textw/netconfig_text.py:280
msgid "Error configuring network device"
msgstr "Hiba történt a hálózati eszköz beállítása közben."

#: textw/netconfig_text.py:280
#, python-format
msgid "Error configuring network device %s"
msgstr "%s hálózati eszközt beállítván hiba történt"

#: textw/partition_text.py:58
msgid "Partitioning Type"
msgstr "Felosztás típusa"

#: textw/partition_text.py:60
msgid "Installation requires partitioning of your hard drive.  The default layout is suitable for most users.  Select what space to use and which drives to use as the install target."
msgstr "A telepítés a merevlemez felosztását igényli.  Az alapértelmezett felosztás megfelel a legtöbb használónak.  Válassza ki, mely helyet használjuk, és mely eszközökre telepítsünk."

#: textw/partition_text.py:80
msgid "Which drive(s) do you want to use for this installation?"
msgstr "Mely eszközöket szeretné használni a telepítéshez?"

#: textw/partition_text.py:95
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr "<Szóköz>,<+>,<->: kijelölés | <F2>: eszköz hozzáadás | <F12>: tovább"

#: textw/partition_text.py:173
#: ui/adddrive.glade.h:4
msgid "Advanced Storage Options"
msgstr "További tároló lehetőségek"

#: textw/partition_text.py:174
#: ui/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr "Hogyan szeretné módosítani az eszköz beállítást?"

#: textw/partition_text.py:203
msgid "Add FCP Device"
msgstr "FCP eszköz hozzáadása"

#: textw/partition_text.py:204
#: ui/zfcp-config.glade.h:5
msgid "zSeries machines can access industry-standard SCSI devices via Fibre Channel (FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr "A zSeries számítógépek hozzá tudnak férni Fibre Channel-ön (FCP) keresztül a szabványos SCSI-eszközökhöz. A következőket kell megadni minden eszközhöz: 16 bites eszközszám, 64 bites egyedi portnév (World Wide Port Name, WWPN), 64 bites FCP LUN."

#: textw/partition_text.py:223
msgid "Add FCoE SAN"
msgstr "FCoE SAN hozzáadása"

#: textw/partition_text.py:224
msgid "Enter the device name for the NIC which is connected to the FCoE SAN. For example \"eth0\"."
msgstr "Adja meg a FCoE SAN-hoz kapcsolódó hálókártya eszköz nevét. Például „eth0”."

#: textw/partition_text.py:225
msgid "NIC device name"
msgstr "Hálókártya név"

#: textw/partition_text.py:232
#, python-format
msgid "%s is not a valid NIC device name."
msgstr "%s nem érvényes hálókártya név."

#: textw/partition_text.py:248
#: ui/iscsi-config.glade.h:7
msgid "Configure iSCSI Parameters"
msgstr "iSCSI paraméterek beállítása"

#: textw/partition_text.py:249
#: ui/iscsi-config.glade.h:8
msgid "To use iSCSI disks, you must provide the address of your iSCSI target and the iSCSI initiator name you've configured for your host."
msgstr "Az iSCSI lemezek használatához meg kell adnia az iSCSI cél eszköz címét és az iSCSI kezdeményező nevét, amit a számítógépen beállított."

#: textw/partition_text.py:250
msgid "Target IP Address"
msgstr "Cél IP cím"

#: textw/partition_text.py:251
msgid "iSCSI Initiator Name"
msgstr "iSCSI kezdeményező neve"

#: textw/partition_text.py:252
msgid "CHAP username"
msgstr "CHAP használó"

#: textw/partition_text.py:253
msgid "CHAP password"
msgstr "CHAP jelszó"

#: textw/partition_text.py:254
msgid "Reverse CHAP username"
msgstr "Inverz CHAP használó"

#: textw/partition_text.py:255
msgid "Reverse CHAP password"
msgstr "Inverz CHAP jelszó"

#: textw/progress_text.py:46
msgid "Package Installation"
msgstr "Csomagok telepítése"

#: textw/timezone_text.py:74
msgid "In which time zone are you located?"
msgstr "Melyik időzónában tartózkodik?"

#: textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr "UTC rendszer óra"

#: textw/upgrade_bootloader_text.py:116
#: textw/upgrade_bootloader_text.py:125
msgid "Update boot loader configuration"
msgstr "Rendszerbetöltő beállítás frissítése"

#: textw/upgrade_bootloader_text.py:128
msgid "Skip boot loader updating"
msgstr "Rendszerbetöltő frissítés kihagyása"

#: textw/upgrade_bootloader_text.py:130
msgid "Create new boot loader configuration"
msgstr "Új rendszerbetöltő beállítás megadása"

#: textw/upgrade_text.py:38
#, python-format
msgid ""
"This release of %s supports the an updated file system, which has several benefits over the file system traditionally shipped in %s.  This installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"%s jelen kiadás támogat egy frissebb fájlrendszert, ami több szempontból is előnyösebb hagyományosan %s rendszerekben adottaknál.  Ez a telepítő képes a kialakított részeket adatvesztés nélkül átalakítani.\n"
"\n"
"Mely részeket szeretné átalakítani?"

#: textw/upgrade_text.py:119
msgid "Free Space"
msgstr "Szabad hely"

#: textw/upgrade_text.py:137
msgid "RAM detected (MB):"
msgstr "Érzékelt memória (MB):"

#: textw/upgrade_text.py:140
msgid "Suggested size (MB):"
msgstr "Javasolt méret (MB):"

#: textw/upgrade_text.py:143
msgid "Swap file size (MB):"
msgstr "Lapozó fájl méret (MB):"

#: textw/upgrade_text.py:151
msgid "Add Swap"
msgstr "Lapozó hozzáadása"

#: textw/upgrade_text.py:176
msgid "The value you entered is not a valid number."
msgstr "A megadott érték nem érvényes szám."

#: textw/upgrade_text.py:209
msgid "Reinstall System"
msgstr "Rendszer újratelepítése"

#: textw/upgrade_text.py:219
msgid "System to Upgrade"
msgstr "Frissitendő rendszer"

#: textw/upgrade_text.py:220
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly install your system."
msgstr ""
"Legalább egy létező Linux-rendszer található a gépen.\n"
"\n"
"Válassza ki a felfejlesztendőt, vagy friss rendszer telepítéséhez válassza „Rendszer újratelepítése” lehetőséget."

#: textw/userauth_text.py:30
msgid "Root Password"
msgstr "Rendszergazda jelszó"

#: textw/userauth_text.py:33
msgid "Pick a root password. You must type it twice to ensure you know it and do not make a typing mistake. "
msgstr "Válasszon rendszergazda jelszót. A jelszót kétszer kell beírnia, hogy biztosan tudja, és ne gépelje el."

#: textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr "Rendszergazda jelszó legalább 6 jel hosszú kell legyen."

#: textw/userauth_text.py:84
#, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"Gyenge jelszót adott meg: %s\n"
"\n"
"Ezzel a jelszóval szeretné folytatni?"

#: textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "%s"

#: textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"Üdvözöli %s!\n"
"\n"

#: textw/zipl_text.py:35
msgid "The z/IPL Boot Loader will be installed on your system after installation is complete. You can now enter any additional kernel and chandev parameters which your machine or your setup require."
msgstr "A z/IPL rendszerindító lesz feltelepítve a telepítés végén. Most megadhatók a gép illetve a konfiguráció működéséhez szükséges további rendszermag- és chandev-paraméterek."

#: textw/zipl_text.py:67
msgid "z/IPL Configuration"
msgstr "z/IPL beállítás"

#: textw/zipl_text.py:75
#: textw/zipl_text.py:79
msgid "Chandev line "
msgstr "Chandev sor "

#: liveinst/liveinst.desktop.in.h:1
msgid "Install"
msgstr "Telepítés"

#: liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr "Live CD telepítése a merevlemezre"

#: liveinst/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr "Telepítés merevlemezre"

#: ui/GroupSelector.glade.h:1
msgid "Some packages associated with this group are not required to be installed but may provide additional functionality.  Please choose the packages which you would like to have installed."
msgstr "E csoportban néhány csomagot nem szükséges telepíteni, de további szolgáltatásokat tehetnek lehetővé.  Kérem válassza ki a telepíteni kívánt csomagokat."

#: ui/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr "Ki_húzás"

#: ui/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr "Minden választható csomag ki_húzása"

#: ui/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr "_Választható csomagok"

#: ui/GroupSelector.glade.h:5
msgid "_Select"
msgstr "_Bejelölés"

#: ui/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr "_Minden választható csomag megjelölése"

#: ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr "dialog1"

#: ui/account.glade.h:1
msgid "Confirm:"
msgstr "Megerősítés:"

#: ui/account.glade.h:2
msgid "Root Password:"
msgstr "Rendszergazda jelszó:"

#: ui/account.glade.h:3
msgid "The root account is used for administering the system.  Enter a password for the root user."
msgstr "A root számlát rendszer adminisztrációra használják.  Adja jelszót a root használónak."

#: ui/adddrive.glade.h:1
msgid "Add _FCoE SAN"
msgstr "_FCoE SAN hozzáadása"

#: ui/adddrive.glade.h:2
msgid "Add _ZFCP LUN"
msgstr "_ZFCP LUN hozzáadása"

#: ui/adddrive.glade.h:3
msgid "Add _iSCSI target"
msgstr "_iSCSI cél hozzáadása"

#: ui/adddrive.glade.h:6
msgid "_Add drive"
msgstr "_Vezérlő hozzáadása"

#: ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr "<b>Tároló _név:</b>"

#: ui/addrepo.glade.h:2
msgid "<b>Repository _type:</b>"
msgstr "<b>Tároló _típus:</b>"

#: ui/addrepo.glade.h:4
msgid "Configure _proxy"
msgstr "_Proxy beállítás"

#: ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Merevlemez"

#: ui/addrepo.glade.h:9
msgid "Please provide the configuration information for this software repository."
msgstr "Kérem, adja meg beállítás információkat e szoftver tárolóhoz."

#: ui/addrepo.glade.h:10
msgid "Proxy U_RL (host:port)"
msgstr "Átjátszó URL (gazda:kapu)"

#: ui/addrepo.glade.h:11
msgid "Proxy pass_word"
msgstr "Proxy jel_szó"

#: ui/addrepo.glade.h:12
msgid "Proxy u_sername"
msgstr "Proxy _használó"

#: ui/addrepo.glade.h:13
msgid "Repository _URL"
msgstr "Tároló _URL"

#: ui/addrepo.glade.h:14
msgid "Select A Directory"
msgstr "Mappa választás"

#: ui/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr "URL egy _tükör lista"

#: ui/addrepo.glade.h:16
msgid "_Directory"
msgstr "_Mappa"

#: ui/addrepo.glade.h:17
msgid "_Options"
msgstr "_Beállítás"

#: ui/addrepo.glade.h:18
msgid "_Partition"
msgstr "_Rész"

#: ui/addrepo.glade.h:19
msgid "_Path"
msgstr "_Út"

#: ui/addrepo.glade.h:20
msgid "_Server"
msgstr "_Kiszolgáló"

#: ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr "Ú_jraindítás"

#: ui/anaconda.glade.h:3
msgid "_Next"
msgstr "_Tovább"

#: ui/autopart.glade.h:1
msgid "<b>Shrink _target (in MB):</b>"
msgstr "<b>_Cél csökkentése (MB-ban):</b>"

#: ui/autopart.glade.h:2
msgid "Installation requires partitioning of your hard drive.  The default layout is suitable for most users.  Select what space to use and which drives to use as the install target. You can also choose to create your own custom layout."
msgstr "A telepítés a merevlemez particionálását igényli.  Az alapértelmezett felosztás megfelel a legtöbb használónak.  Válassza ki, mely helyet használjuk, és mely eszközökre telepítsünk. Választhatja azt is, hogy készít egy saját egyedi felosztást."

#: ui/autopart.glade.h:3
msgid "Re_view and modify partitioning layout"
msgstr "Felosztás felül_vizsgálata és megváltoztatása"

#: ui/autopart.glade.h:4
msgid "Volume to Shrink"
msgstr "Csökkentendő kötet"

#: ui/autopart.glade.h:5
msgid "What drive would you like to _boot this installation from?"
msgstr "Melyik eszközről szeretné in_dítani e telepítést?"

#: ui/autopart.glade.h:6
msgid "Which partition would you like to shrink to make room for your installation?"
msgstr "Melyik részt szeretné csökkenteni, hogy helyet készítsen a telepítésnek?"

#: ui/autopart.glade.h:7
msgid "_Advanced storage configuration"
msgstr "_Haladó tároló eszköz beállítás"

#: ui/autopart.glade.h:8
msgid "_Encrypt system"
msgstr "Rendszer _titkosítása"

#: ui/autopart.glade.h:9
msgid "_Select the drive(s) to use for this installation."
msgstr "_Válassza ki a telepítéshez használt eszközöket."

#: ui/autopart.glade.h:10
msgid "_Shrink"
msgstr "Z_sugorít"

#: ui/blwhere.glade.h:1
msgid "/boot"
msgstr "/boot"

#: ui/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr "BIOS eszköz sorrend"

#: ui/blwhere.glade.h:3
msgid "Boot loader device"
msgstr "Rendszerindító eszköz"

#: ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr "Első BIOS eszköz:"

#: ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr "Negyedik BIOS eszköz:"

#: ui/blwhere.glade.h:6
msgid "MBR"
msgstr "MBR"

#: ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr "Második BIOS eszköz:"

#: ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr "Harmadik BIOS eszköz:"

#: ui/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr "Hova szeretné telepíteni a rendszerindítót?"

#: ui/detailed-dialog.glade.h:1
msgid "Info"
msgstr "Infó"

#: ui/detailed-dialog.glade.h:2
msgid "_Details"
msgstr "_Részletek"

#: ui/fcoe-config.glade.h:1
msgid "Configure FCoE Parameters"
msgstr "FCoE paraméterek beállítása"

#: ui/fcoe-config.glade.h:2
msgid "NIC"
msgstr "NIC"

#: ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""
"Kérem válassza ki a hálózati csatlakozót ami kapcsolódik\n"
"az ö FCoE switch-hez."

#: ui/fcoe-config.glade.h:5
msgid "_Add FCoE Disk(s)"
msgstr "FCoE lemez(ek) hozzá_adása"

#: ui/instkey.glade.h:2
#, no-c-format
msgid "%(instkey)s:"
msgstr "%(instkey)s:"

#: ui/instkey.glade.h:4
#, no-c-format
msgid "Please enter your %(instkey)s."
msgstr "Adja meg ezt: %(instkey)s"

#: ui/iscsi-config.glade.h:1
msgid "<b>CHAP _Password:</b>"
msgstr "<b>CHAP _Jelszó:</b>"

#: ui/iscsi-config.glade.h:2
msgid "<b>CHAP _Username:</b>"
msgstr "<b>CHAP _Használónév:</b>"

#: ui/iscsi-config.glade.h:3
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr "<b>Inverz CHAP J_elszó:</b>"

#: ui/iscsi-config.glade.h:4
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr "<b>Inverz CHAP H_asználónév:</b>"

#: ui/iscsi-config.glade.h:5
msgid "<b>_Target IP Address:</b>"
msgstr "<b>_Cél IP cím:</b>"

#: ui/iscsi-config.glade.h:6
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr "<b>iSCSI kezdeményező _név:</b>"

#: ui/iscsi-config.glade.h:9
msgid "_Add target"
msgstr "_Cél hozzáadása"

#: ui/lukspassphrase.glade.h:1
msgid "Also add this passphrase to all existing encrypted devices to streamline the boot process"
msgstr "E jelszó használata minden meglévő titkosított eszközhöz, hogy meggyorsítsuk a betöltés folyamatát"

#: ui/lukspassphrase.glade.h:2
msgid "Choose a passphrase for this encrypted partition. You will be prompted for the passphrase during system boot."
msgstr "Válasszon jelszót e titkosított résznek. A rendszerindító induláskor kérni fogja ezt a jelszót."

#: ui/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr "Jelszó megerősítése:"

#: ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr "Adja meg a titkosított rész jelszavát"

#: ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr "Jelszó megadása:"

#: ui/netconfig.glade.h:2
msgid "<b>Gateway:</b>"
msgstr "<b>Átjáró:</b>"

#: ui/netconfig.glade.h:3
msgid "<b>IPv4 Address:</b>"
msgstr "<b>IPv4 cím:</b>"

#: ui/netconfig.glade.h:4
msgid "<b>IPv6 Address:</b>"
msgstr "<b>IPv6 cím:</b>"

#: ui/netconfig.glade.h:5
msgid "<b>Nameserver:</b>"
msgstr "<b>Névkiszolgáló:</b>"

#: ui/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr "<b>_Csatoló:</b>"

#: ui/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr "IPv_4 támogatás engedélyezése"

#: ui/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr "IPv_6 támogatás engedélyezése"

#: ui/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr "_Változó IP beállítás (DHCP)"

#: ui/network.glade.h:1
msgid "Hostname:"
msgstr "Gazdanév:"

#: ui/network.glade.h:2
msgid "Please name this computer.  The hostname identifies the computer on a network."
msgstr "Kérem nevezze el e számítógépet.  Hálózatban a gazda név azonosítja a számítógépet."

#: ui/tasksel.glade.h:1
msgid "Customize _later"
msgstr "Testreszabás _később"

#: ui/tasksel.glade.h:2
msgid "Please select any additional repositories that you want to use for software installation."
msgstr "Kérem válasszon ki minden további tárolót, amit használni akar a szoftver telepítéséhez."

#: ui/tasksel.glade.h:4
#, no-c-format
msgid "The default installation of %s includes a set of software applicable for general internet usage.  What additional tasks would you like your system to support?"
msgstr "%s alapértelmezett telepítése tartalmaz egy általános Internet használathoz készült szoftver csoportot.  Milyen további feladatokat szeretne, hogy a rendszer támogassa?"

#: ui/tasksel.glade.h:5
msgid "You can further customize the software selection now, or after install via the software management application."
msgstr "Még tovább testreszabhatja a szoftver válogatást most vagy később a telepítés után a szoftverkezelő alkalmazással."

#: ui/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr "További szoftver tárolók _hozzáadása"

#: ui/tasksel.glade.h:7
msgid "_Customize now"
msgstr "_Testreszabás most"

#: ui/tasksel.glade.h:8
msgid "_Modify repository"
msgstr "_Tároló megváltoztatása"

#: ui/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr "<b>Eszköz szám:</b>"

#: ui/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr "<b>FCP LUN:</b>"

#: ui/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr "<b>WWPN:</b>"

#: ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr "FCP eszköz hozzáadása"

#: lang-table.h:1
msgid "Afrikaans"
msgstr "Afrikai"

#: lang-table.h:2
msgid "Arabic"
msgstr "Arab"

#: lang-table.h:3
msgid "Assamese"
msgstr "Assami"

#: lang-table.h:4
msgid "Bengali"
msgstr "Bengáli"

#: lang-table.h:5
msgid "Bengali(India)"
msgstr "Bengáli (India)"

#: lang-table.h:6
msgid "Bulgarian"
msgstr "Bulgár"

#: lang-table.h:7
msgid "Catalan"
msgstr "Katalán"

#: lang-table.h:8
msgid "Chinese(Simplified)"
msgstr "Chinese (Egyszerű)"

#: lang-table.h:9
msgid "Chinese(Traditional)"
msgstr "Kínai (Hagyományos)"

#: lang-table.h:10
msgid "Croatian"
msgstr "Horvát"

#: lang-table.h:11
msgid "Czech"
msgstr "Cseh"

#: lang-table.h:12
msgid "Danish"
msgstr "Dán"

#: lang-table.h:13
msgid "Dutch"
msgstr "Holland"

#: lang-table.h:14
msgid "English"
msgstr "Angol"

#: lang-table.h:15
msgid "Estonian"
msgstr "Észt"

#: lang-table.h:16
msgid "Finnish"
msgstr "Finn"

#: lang-table.h:17
msgid "French"
msgstr "Francia"

#: lang-table.h:18
msgid "German"
msgstr "Német"

#: lang-table.h:19
msgid "Greek"
msgstr "Görög"

#: lang-table.h:20
msgid "Gujarati"
msgstr "Gujarati"

#: lang-table.h:21
msgid "Hebrew"
msgstr "Héber"

#: lang-table.h:22
msgid "Hindi"
msgstr "Hindi"

#: lang-table.h:23
msgid "Hungarian"
msgstr "Magyar"

#: lang-table.h:24
msgid "Icelandic"
msgstr "Izlandi"

#: lang-table.h:25
msgid "Iloko"
msgstr "Iloko"

#: lang-table.h:26
msgid "Indonesian"
msgstr "Indonéz"

#: lang-table.h:27
msgid "Italian"
msgstr "Olasz"

#: lang-table.h:28
msgid "Japanese"
msgstr "Japán"

#: lang-table.h:29
msgid "Kannada"
msgstr "Kannada"

#: lang-table.h:30
msgid "Korean"
msgstr "Koreai"

#: lang-table.h:31
msgid "Macedonian"
msgstr "Macedón"

#: lang-table.h:32
msgid "Maithili"
msgstr "Maithili"

#: lang-table.h:33
msgid "Malay"
msgstr "Maláj"

#: lang-table.h:34
msgid "Malayalam"
msgstr "Malayalam-i"

#: lang-table.h:35
msgid "Marathi"
msgstr "Maráthi"

#: lang-table.h:36
msgid "Nepali"
msgstr "Nepál"

#: lang-table.h:37
msgid "Norwegian(Bokmål)"
msgstr "Norvég (Bokmål)"

#: lang-table.h:38
msgid "Northern Sotho"
msgstr "Északi Sotho"

#: lang-table.h:39
msgid "Oriya"
msgstr "Oriya"

#: lang-table.h:40
msgid "Persian"
msgstr "Perzsa"

#: lang-table.h:41
msgid "Polish"
msgstr "Lengyel"

#: lang-table.h:42
msgid "Portuguese"
msgstr "Portugál"

#: lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr "Portugál (Brazil)"

#: lang-table.h:44
msgid "Punjabi"
msgstr "India"

#: lang-table.h:45
msgid "Romanian"
msgstr "Román"

#: lang-table.h:46
msgid "Russian"
msgstr "Orosz"

#: lang-table.h:47
msgid "Serbian"
msgstr "Szerb"

#: lang-table.h:48
msgid "Serbian(Latin)"
msgstr "Szerb (Latin)"

#: lang-table.h:49
msgid "Sinhala"
msgstr "Sinhala"

#: lang-table.h:50
msgid "Slovak"
msgstr "Szlovák"

#: lang-table.h:51
msgid "Slovenian"
msgstr "Szlovén"

#: lang-table.h:52
msgid "Spanish"
msgstr "Spanyol"

#: lang-table.h:53
msgid "Swedish"
msgstr "Svéd"

#: lang-table.h:54
msgid "Tajik"
msgstr "Thai"

#: lang-table.h:55
msgid "Tamil"
msgstr "Tamil"

#: lang-table.h:56
msgid "Telugu"
msgstr "Telugu"

#: lang-table.h:57
msgid "Turkish"
msgstr "Török"

#: lang-table.h:58
msgid "Ukrainian"
msgstr "Ukrán"

#: lang-table.h:59
msgid "Vietnamese"
msgstr "Vietnam"

#: lang-table.h:60
msgid "Welsh"
msgstr "Welsh"

#: lang-table.h:61
msgid "Zulu"
msgstr "Zulu"

#~ msgid ""
#~ "The following error was found while parsing thekickstart configuration "
#~ "file:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "A kickstart beállítás fájlt értelmezvén a következő hiba merült fel:\n"
#~ "\n"
#~ "%s"
#~ msgid "attr dict must include a type"
#~ msgstr "attr paraméterben meg kell adni a típust"
#~ msgid "window1"
#~ msgstr "window1"
#~ msgid "Disable _dmraid device"
#~ msgstr "A _dmraid eszköz letiltása"
#~ msgid "label1"
#~ msgstr "label1"

